From 161d708fccad035e224a3468f39e8552c1999661 Mon Sep 17 00:00:00 2001
From: Steve Ellcey <sellcey at caviumnetworks dot com>
Date: Wed, 22 Feb 2017 08:29:00 -0800
Subject: [PATCH 1/2] Patch 1 of 2 for ILP32 aarch64

This is the first of two aarch64 ILP32 patches.  I think the only
blocker in checking these in is getting the ILP32 kernel changes
accepted and that is currently being worked on.   In the meantime I
thought I would repost them because the previous patches do not apply
cleanly to the top-of-tree anymore and these will.  This first patch is
just the abilist file changes.  The version numbers are still at 2.25,
I don't know if that is going to change for the next glibc release but
this matches what is currently at top-of-tree.

Steve Ellcey
sellcey@cavium.com

2017-02-22  Andrew Pinski  <andrew.pinski@caviumnetworks.com>
	    Yury Norov  <ynorov@caviumnetworks.com>
	    Steve Ellcey  <sellcey@caviumnetworks.com>

	* sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist: New file.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist:
	Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libcrypt.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/ld.abilist: Moved to lp64 directory.
	* sysdeps/unix/sysv/linux/aarch64/libBrokenLocale.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libanl.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libc.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libcrypt.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libdl.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libm.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libnsl.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libpthread.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libresolv.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/librt.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libthread_db.abilist: Likewise.
	* sysdeps/unix/sysv/linux/aarch64/libutil.abilist: Likewise.
---
 sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist   |   10 +
 .../linux/aarch64/ilp32/libBrokenLocale.abilist    |    2 +
 .../unix/sysv/linux/aarch64/ilp32/libanl.abilist   |    5 +
 sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist | 2063 ++++++++++++++++++++
 .../unix/sysv/linux/aarch64/ilp32/libcrypt.abilist |    8 +
 .../unix/sysv/linux/aarch64/ilp32/libdl.abilist    |   10 +
 sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist |  455 +++++
 .../unix/sysv/linux/aarch64/ilp32/libnsl.abilist   |  122 ++
 .../sysv/linux/aarch64/ilp32/libpthread.abilist    |  218 +++
 .../sysv/linux/aarch64/ilp32/libresolv.abilist     |   82 +
 .../unix/sysv/linux/aarch64/ilp32/librt.abilist    |   36 +
 .../sysv/linux/aarch64/ilp32/libthread_db.abilist  |   41 +
 .../unix/sysv/linux/aarch64/ilp32/libutil.abilist  |    7 +
 .../unix/sysv/linux/aarch64/{ => lp64}/ld.abilist  |    0
 .../aarch64/{ => lp64}/libBrokenLocale.abilist     |    0
 .../sysv/linux/aarch64/{ => lp64}/libanl.abilist   |    0
 .../sysv/linux/aarch64/{ => lp64}/libc.abilist     |    0
 .../sysv/linux/aarch64/{ => lp64}/libcrypt.abilist |    0
 .../sysv/linux/aarch64/{ => lp64}/libdl.abilist    |    0
 .../sysv/linux/aarch64/{ => lp64}/libm.abilist     |    0
 .../sysv/linux/aarch64/{ => lp64}/libnsl.abilist   |    0
 .../linux/aarch64/{ => lp64}/libpthread.abilist    |    0
 .../linux/aarch64/{ => lp64}/libresolv.abilist     |    0
 .../sysv/linux/aarch64/{ => lp64}/librt.abilist    |    0
 .../linux/aarch64/{ => lp64}/libthread_db.abilist  |    0
 .../sysv/linux/aarch64/{ => lp64}/libutil.abilist  |    0
 26 files changed, 3059 insertions(+)
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libcrypt.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist
 create mode 100644 sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/ld.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libBrokenLocale.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libanl.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libc.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libcrypt.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libdl.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libm.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libnsl.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libpthread.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libresolv.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/librt.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libthread_db.abilist (100%)
 rename sysdeps/unix/sysv/linux/aarch64/{ => lp64}/libutil.abilist (100%)

diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist
new file mode 100644
index 0000000000..1660a2607e
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/ld.abilist
@@ -0,0 +1,10 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 __libc_stack_end D 0x4
+GLIBC_2.25 __stack_chk_guard D 0x4
+GLIBC_2.25 __tls_get_addr F
+GLIBC_2.25 _dl_mcount F
+GLIBC_2.25 _r_debug D 0x14
+GLIBC_2.25 calloc F
+GLIBC_2.25 free F
+GLIBC_2.25 malloc F
+GLIBC_2.25 realloc F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist
new file mode 100644
index 0000000000..3d89dda2cc
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libBrokenLocale.abilist
@@ -0,0 +1,2 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 __ctype_get_mb_cur_max F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist
new file mode 100644
index 0000000000..992a8dbab4
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libanl.abilist
@@ -0,0 +1,5 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 gai_cancel F
+GLIBC_2.25 gai_error F
+GLIBC_2.25 gai_suspend F
+GLIBC_2.25 getaddrinfo_a F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist
new file mode 100644
index 0000000000..fd5bdb4b94
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libc.abilist
@@ -0,0 +1,2063 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 _Exit F
+GLIBC_2.25 _IO_2_1_stderr_ D 0xa0
+GLIBC_2.25 _IO_2_1_stdin_ D 0xa0
+GLIBC_2.25 _IO_2_1_stdout_ D 0xa0
+GLIBC_2.25 _IO_adjust_column F
+GLIBC_2.25 _IO_adjust_wcolumn F
+GLIBC_2.25 _IO_default_doallocate F
+GLIBC_2.25 _IO_default_finish F
+GLIBC_2.25 _IO_default_pbackfail F
+GLIBC_2.25 _IO_default_uflow F
+GLIBC_2.25 _IO_default_xsgetn F
+GLIBC_2.25 _IO_default_xsputn F
+GLIBC_2.25 _IO_do_write F
+GLIBC_2.25 _IO_doallocbuf F
+GLIBC_2.25 _IO_fclose F
+GLIBC_2.25 _IO_fdopen F
+GLIBC_2.25 _IO_feof F
+GLIBC_2.25 _IO_ferror F
+GLIBC_2.25 _IO_fflush F
+GLIBC_2.25 _IO_fgetpos F
+GLIBC_2.25 _IO_fgetpos64 F
+GLIBC_2.25 _IO_fgets F
+GLIBC_2.25 _IO_file_attach F
+GLIBC_2.25 _IO_file_close F
+GLIBC_2.25 _IO_file_close_it F
+GLIBC_2.25 _IO_file_doallocate F
+GLIBC_2.25 _IO_file_finish F
+GLIBC_2.25 _IO_file_fopen F
+GLIBC_2.25 _IO_file_init F
+GLIBC_2.25 _IO_file_jumps D 0x54
+GLIBC_2.25 _IO_file_open F
+GLIBC_2.25 _IO_file_overflow F
+GLIBC_2.25 _IO_file_read F
+GLIBC_2.25 _IO_file_seek F
+GLIBC_2.25 _IO_file_seekoff F
+GLIBC_2.25 _IO_file_setbuf F
+GLIBC_2.25 _IO_file_stat F
+GLIBC_2.25 _IO_file_sync F
+GLIBC_2.25 _IO_file_underflow F
+GLIBC_2.25 _IO_file_write F
+GLIBC_2.25 _IO_file_xsputn F
+GLIBC_2.25 _IO_flockfile F
+GLIBC_2.25 _IO_flush_all F
+GLIBC_2.25 _IO_flush_all_linebuffered F
+GLIBC_2.25 _IO_fopen F
+GLIBC_2.25 _IO_fprintf F
+GLIBC_2.25 _IO_fputs F
+GLIBC_2.25 _IO_fread F
+GLIBC_2.25 _IO_free_backup_area F
+GLIBC_2.25 _IO_free_wbackup_area F
+GLIBC_2.25 _IO_fsetpos F
+GLIBC_2.25 _IO_fsetpos64 F
+GLIBC_2.25 _IO_ftell F
+GLIBC_2.25 _IO_ftrylockfile F
+GLIBC_2.25 _IO_funlockfile F
+GLIBC_2.25 _IO_fwrite F
+GLIBC_2.25 _IO_getc F
+GLIBC_2.25 _IO_getline F
+GLIBC_2.25 _IO_getline_info F
+GLIBC_2.25 _IO_gets F
+GLIBC_2.25 _IO_init F
+GLIBC_2.25 _IO_init_marker F
+GLIBC_2.25 _IO_init_wmarker F
+GLIBC_2.25 _IO_iter_begin F
+GLIBC_2.25 _IO_iter_end F
+GLIBC_2.25 _IO_iter_file F
+GLIBC_2.25 _IO_iter_next F
+GLIBC_2.25 _IO_least_wmarker F
+GLIBC_2.25 _IO_link_in F
+GLIBC_2.25 _IO_list_all D 0x4
+GLIBC_2.25 _IO_list_lock F
+GLIBC_2.25 _IO_list_resetlock F
+GLIBC_2.25 _IO_list_unlock F
+GLIBC_2.25 _IO_marker_delta F
+GLIBC_2.25 _IO_marker_difference F
+GLIBC_2.25 _IO_padn F
+GLIBC_2.25 _IO_peekc_locked F
+GLIBC_2.25 _IO_popen F
+GLIBC_2.25 _IO_printf F
+GLIBC_2.25 _IO_proc_close F
+GLIBC_2.25 _IO_proc_open F
+GLIBC_2.25 _IO_putc F
+GLIBC_2.25 _IO_puts F
+GLIBC_2.25 _IO_remove_marker F
+GLIBC_2.25 _IO_seekmark F
+GLIBC_2.25 _IO_seekoff F
+GLIBC_2.25 _IO_seekpos F
+GLIBC_2.25 _IO_seekwmark F
+GLIBC_2.25 _IO_setb F
+GLIBC_2.25 _IO_setbuffer F
+GLIBC_2.25 _IO_setvbuf F
+GLIBC_2.25 _IO_sgetn F
+GLIBC_2.25 _IO_sprintf F
+GLIBC_2.25 _IO_sputbackc F
+GLIBC_2.25 _IO_sputbackwc F
+GLIBC_2.25 _IO_sscanf F
+GLIBC_2.25 _IO_str_init_readonly F
+GLIBC_2.25 _IO_str_init_static F
+GLIBC_2.25 _IO_str_overflow F
+GLIBC_2.25 _IO_str_pbackfail F
+GLIBC_2.25 _IO_str_seekoff F
+GLIBC_2.25 _IO_str_underflow F
+GLIBC_2.25 _IO_sungetc F
+GLIBC_2.25 _IO_sungetwc F
+GLIBC_2.25 _IO_switch_to_get_mode F
+GLIBC_2.25 _IO_switch_to_main_wget_area F
+GLIBC_2.25 _IO_switch_to_wbackup_area F
+GLIBC_2.25 _IO_switch_to_wget_mode F
+GLIBC_2.25 _IO_un_link F
+GLIBC_2.25 _IO_ungetc F
+GLIBC_2.25 _IO_unsave_markers F
+GLIBC_2.25 _IO_unsave_wmarkers F
+GLIBC_2.25 _IO_vfprintf F
+GLIBC_2.25 _IO_vfscanf F
+GLIBC_2.25 _IO_vsprintf F
+GLIBC_2.25 _IO_wdefault_doallocate F
+GLIBC_2.25 _IO_wdefault_finish F
+GLIBC_2.25 _IO_wdefault_pbackfail F
+GLIBC_2.25 _IO_wdefault_uflow F
+GLIBC_2.25 _IO_wdefault_xsgetn F
+GLIBC_2.25 _IO_wdefault_xsputn F
+GLIBC_2.25 _IO_wdo_write F
+GLIBC_2.25 _IO_wdoallocbuf F
+GLIBC_2.25 _IO_wfile_jumps D 0x54
+GLIBC_2.25 _IO_wfile_overflow F
+GLIBC_2.25 _IO_wfile_seekoff F
+GLIBC_2.25 _IO_wfile_sync F
+GLIBC_2.25 _IO_wfile_underflow F
+GLIBC_2.25 _IO_wfile_xsputn F
+GLIBC_2.25 _IO_wmarker_delta F
+GLIBC_2.25 _IO_wsetb F
+GLIBC_2.25 ___brk_addr D 0x4
+GLIBC_2.25 __adjtimex F
+GLIBC_2.25 __after_morecore_hook D 0x4
+GLIBC_2.25 __argz_count F
+GLIBC_2.25 __argz_next F
+GLIBC_2.25 __argz_stringify F
+GLIBC_2.25 __asprintf F
+GLIBC_2.25 __asprintf_chk F
+GLIBC_2.25 __assert F
+GLIBC_2.25 __assert_fail F
+GLIBC_2.25 __assert_perror_fail F
+GLIBC_2.25 __backtrace F
+GLIBC_2.25 __backtrace_symbols F
+GLIBC_2.25 __backtrace_symbols_fd F
+GLIBC_2.25 __bsd_getpgrp F
+GLIBC_2.25 __bzero F
+GLIBC_2.25 __check_rhosts_file D 0x4
+GLIBC_2.25 __chk_fail F
+GLIBC_2.25 __clone F
+GLIBC_2.25 __close F
+GLIBC_2.25 __cmsg_nxthdr F
+GLIBC_2.25 __confstr_chk F
+GLIBC_2.25 __connect F
+GLIBC_2.25 __ctype_b_loc F
+GLIBC_2.25 __ctype_get_mb_cur_max F
+GLIBC_2.25 __ctype_tolower_loc F
+GLIBC_2.25 __ctype_toupper_loc F
+GLIBC_2.25 __curbrk D 0x4
+GLIBC_2.25 __cxa_at_quick_exit F
+GLIBC_2.25 __cxa_atexit F
+GLIBC_2.25 __cxa_finalize F
+GLIBC_2.25 __cxa_thread_atexit_impl F
+GLIBC_2.25 __cyg_profile_func_enter F
+GLIBC_2.25 __cyg_profile_func_exit F
+GLIBC_2.25 __daylight D 0x4
+GLIBC_2.25 __dcgettext F
+GLIBC_2.25 __default_morecore F
+GLIBC_2.25 __dgettext F
+GLIBC_2.25 __dprintf_chk F
+GLIBC_2.25 __dup2 F
+GLIBC_2.25 __duplocale F
+GLIBC_2.25 __endmntent F
+GLIBC_2.25 __environ D 0x4
+GLIBC_2.25 __errno_location F
+GLIBC_2.25 __explicit_bzero_chk F
+GLIBC_2.25 __fbufsize F
+GLIBC_2.25 __fcntl F
+GLIBC_2.25 __fdelt_chk F
+GLIBC_2.25 __fdelt_warn F
+GLIBC_2.25 __ffs F
+GLIBC_2.25 __fgets_chk F
+GLIBC_2.25 __fgets_unlocked_chk F
+GLIBC_2.25 __fgetws_chk F
+GLIBC_2.25 __fgetws_unlocked_chk F
+GLIBC_2.25 __finite F
+GLIBC_2.25 __finitef F
+GLIBC_2.25 __finitel F
+GLIBC_2.25 __flbf F
+GLIBC_2.25 __fork F
+GLIBC_2.25 __fpending F
+GLIBC_2.25 __fprintf_chk F
+GLIBC_2.25 __fpu_control D 0x4
+GLIBC_2.25 __fpurge F
+GLIBC_2.25 __fread_chk F
+GLIBC_2.25 __fread_unlocked_chk F
+GLIBC_2.25 __freadable F
+GLIBC_2.25 __freading F
+GLIBC_2.25 __free_hook D 0x4
+GLIBC_2.25 __freelocale F
+GLIBC_2.25 __fsetlocking F
+GLIBC_2.25 __fwprintf_chk F
+GLIBC_2.25 __fwritable F
+GLIBC_2.25 __fwriting F
+GLIBC_2.25 __fxstat F
+GLIBC_2.25 __fxstat64 F
+GLIBC_2.25 __fxstatat F
+GLIBC_2.25 __fxstatat64 F
+GLIBC_2.25 __getauxval F
+GLIBC_2.25 __getcwd_chk F
+GLIBC_2.25 __getdelim F
+GLIBC_2.25 __getdomainname_chk F
+GLIBC_2.25 __getgroups_chk F
+GLIBC_2.25 __gethostname_chk F
+GLIBC_2.25 __getlogin_r_chk F
+GLIBC_2.25 __getmntent_r F
+GLIBC_2.25 __getpagesize F
+GLIBC_2.25 __getpgid F
+GLIBC_2.25 __getpid F
+GLIBC_2.25 __gets_chk F
+GLIBC_2.25 __gettimeofday F
+GLIBC_2.25 __getwd_chk F
+GLIBC_2.25 __gmtime_r F
+GLIBC_2.25 __h_errno_location F
+GLIBC_2.25 __isalnum_l F
+GLIBC_2.25 __isalpha_l F
+GLIBC_2.25 __isascii_l F
+GLIBC_2.25 __isblank_l F
+GLIBC_2.25 __iscntrl_l F
+GLIBC_2.25 __isctype F
+GLIBC_2.25 __isdigit_l F
+GLIBC_2.25 __isgraph_l F
+GLIBC_2.25 __isinf F
+GLIBC_2.25 __isinff F
+GLIBC_2.25 __isinfl F
+GLIBC_2.25 __islower_l F
+GLIBC_2.25 __isnan F
+GLIBC_2.25 __isnanf F
+GLIBC_2.25 __isnanl F
+GLIBC_2.25 __isoc99_fscanf F
+GLIBC_2.25 __isoc99_fwscanf F
+GLIBC_2.25 __isoc99_scanf F
+GLIBC_2.25 __isoc99_sscanf F
+GLIBC_2.25 __isoc99_swscanf F
+GLIBC_2.25 __isoc99_vfscanf F
+GLIBC_2.25 __isoc99_vfwscanf F
+GLIBC_2.25 __isoc99_vscanf F
+GLIBC_2.25 __isoc99_vsscanf F
+GLIBC_2.25 __isoc99_vswscanf F
+GLIBC_2.25 __isoc99_vwscanf F
+GLIBC_2.25 __isoc99_wscanf F
+GLIBC_2.25 __isprint_l F
+GLIBC_2.25 __ispunct_l F
+GLIBC_2.25 __isspace_l F
+GLIBC_2.25 __isupper_l F
+GLIBC_2.25 __iswalnum_l F
+GLIBC_2.25 __iswalpha_l F
+GLIBC_2.25 __iswblank_l F
+GLIBC_2.25 __iswcntrl_l F
+GLIBC_2.25 __iswctype F
+GLIBC_2.25 __iswctype_l F
+GLIBC_2.25 __iswdigit_l F
+GLIBC_2.25 __iswgraph_l F
+GLIBC_2.25 __iswlower_l F
+GLIBC_2.25 __iswprint_l F
+GLIBC_2.25 __iswpunct_l F
+GLIBC_2.25 __iswspace_l F
+GLIBC_2.25 __iswupper_l F
+GLIBC_2.25 __iswxdigit_l F
+GLIBC_2.25 __isxdigit_l F
+GLIBC_2.25 __ivaliduser F
+GLIBC_2.25 __key_decryptsession_pk_LOCAL D 0x4
+GLIBC_2.25 __key_encryptsession_pk_LOCAL D 0x4
+GLIBC_2.25 __key_gendes_LOCAL D 0x4
+GLIBC_2.25 __libc_allocate_rtsig F
+GLIBC_2.25 __libc_calloc F
+GLIBC_2.25 __libc_current_sigrtmax F
+GLIBC_2.25 __libc_current_sigrtmin F
+GLIBC_2.25 __libc_free F
+GLIBC_2.25 __libc_freeres F
+GLIBC_2.25 __libc_init_first F
+GLIBC_2.25 __libc_mallinfo F
+GLIBC_2.25 __libc_malloc F
+GLIBC_2.25 __libc_mallopt F
+GLIBC_2.25 __libc_memalign F
+GLIBC_2.25 __libc_pvalloc F
+GLIBC_2.25 __libc_realloc F
+GLIBC_2.25 __libc_sa_len F
+GLIBC_2.25 __libc_start_main F
+GLIBC_2.25 __libc_valloc F
+GLIBC_2.25 __longjmp_chk F
+GLIBC_2.25 __lseek F
+GLIBC_2.25 __lxstat F
+GLIBC_2.25 __lxstat64 F
+GLIBC_2.25 __malloc_hook D 0x4
+GLIBC_2.25 __mbrlen F
+GLIBC_2.25 __mbrtowc F
+GLIBC_2.25 __mbsnrtowcs_chk F
+GLIBC_2.25 __mbsrtowcs_chk F
+GLIBC_2.25 __mbstowcs_chk F
+GLIBC_2.25 __memalign_hook D 0x4
+GLIBC_2.25 __memcpy_chk F
+GLIBC_2.25 __memmove_chk F
+GLIBC_2.25 __mempcpy F
+GLIBC_2.25 __mempcpy_chk F
+GLIBC_2.25 __memset_chk F
+GLIBC_2.25 __monstartup F
+GLIBC_2.25 __morecore D 0x4
+GLIBC_2.25 __nanosleep F
+GLIBC_2.25 __newlocale F
+GLIBC_2.25 __nl_langinfo_l F
+GLIBC_2.25 __nss_configure_lookup F
+GLIBC_2.25 __nss_database_lookup F
+GLIBC_2.25 __nss_group_lookup F
+GLIBC_2.25 __nss_hostname_digits_dots F
+GLIBC_2.25 __nss_hosts_lookup F
+GLIBC_2.25 __nss_next F
+GLIBC_2.25 __nss_passwd_lookup F
+GLIBC_2.25 __obstack_printf_chk F
+GLIBC_2.25 __obstack_vprintf_chk F
+GLIBC_2.25 __open F
+GLIBC_2.25 __open64 F
+GLIBC_2.25 __open64_2 F
+GLIBC_2.25 __open_2 F
+GLIBC_2.25 __openat64_2 F
+GLIBC_2.25 __openat_2 F
+GLIBC_2.25 __overflow F
+GLIBC_2.25 __pipe F
+GLIBC_2.25 __poll F
+GLIBC_2.25 __poll_chk F
+GLIBC_2.25 __posix_getopt F
+GLIBC_2.25 __ppoll_chk F
+GLIBC_2.25 __pread64 F
+GLIBC_2.25 __pread64_chk F
+GLIBC_2.25 __pread_chk F
+GLIBC_2.25 __printf_chk F
+GLIBC_2.25 __printf_fp F
+GLIBC_2.25 __profile_frequency F
+GLIBC_2.25 __progname D 0x4
+GLIBC_2.25 __progname_full D 0x4
+GLIBC_2.25 __ptsname_r_chk F
+GLIBC_2.25 __pwrite64 F
+GLIBC_2.25 __rawmemchr F
+GLIBC_2.25 __rcmd_errstr D 0x4
+GLIBC_2.25 __read F
+GLIBC_2.25 __read_chk F
+GLIBC_2.25 __readlink_chk F
+GLIBC_2.25 __readlinkat_chk F
+GLIBC_2.25 __realloc_hook D 0x4
+GLIBC_2.25 __realpath_chk F
+GLIBC_2.25 __recv_chk F
+GLIBC_2.25 __recvfrom_chk F
+GLIBC_2.25 __register_atfork F
+GLIBC_2.25 __res_init F
+GLIBC_2.25 __res_nclose F
+GLIBC_2.25 __res_ninit F
+GLIBC_2.25 __res_randomid F
+GLIBC_2.25 __res_state F
+GLIBC_2.25 __rpc_thread_createerr F
+GLIBC_2.25 __rpc_thread_svc_fdset F
+GLIBC_2.25 __rpc_thread_svc_max_pollfd F
+GLIBC_2.25 __rpc_thread_svc_pollfd F
+GLIBC_2.25 __sbrk F
+GLIBC_2.25 __sched_cpualloc F
+GLIBC_2.25 __sched_cpucount F
+GLIBC_2.25 __sched_cpufree F
+GLIBC_2.25 __sched_get_priority_max F
+GLIBC_2.25 __sched_get_priority_min F
+GLIBC_2.25 __sched_getparam F
+GLIBC_2.25 __sched_getscheduler F
+GLIBC_2.25 __sched_setscheduler F
+GLIBC_2.25 __sched_yield F
+GLIBC_2.25 __select F
+GLIBC_2.25 __setmntent F
+GLIBC_2.25 __setpgid F
+GLIBC_2.25 __sigaction F
+GLIBC_2.25 __sigaddset F
+GLIBC_2.25 __sigdelset F
+GLIBC_2.25 __sigismember F
+GLIBC_2.25 __signbit F
+GLIBC_2.25 __signbitf F
+GLIBC_2.25 __signbitl F
+GLIBC_2.25 __sigpause F
+GLIBC_2.25 __sigsetjmp F
+GLIBC_2.25 __sigsuspend F
+GLIBC_2.25 __snprintf_chk F
+GLIBC_2.25 __sprintf_chk F
+GLIBC_2.25 __stack_chk_fail F
+GLIBC_2.25 __statfs F
+GLIBC_2.25 __stpcpy F
+GLIBC_2.25 __stpcpy_chk F
+GLIBC_2.25 __stpncpy F
+GLIBC_2.25 __stpncpy_chk F
+GLIBC_2.25 __strcasecmp F
+GLIBC_2.25 __strcasecmp_l F
+GLIBC_2.25 __strcasestr F
+GLIBC_2.25 __strcat_chk F
+GLIBC_2.25 __strcoll_l F
+GLIBC_2.25 __strcpy_chk F
+GLIBC_2.25 __strdup F
+GLIBC_2.25 __strerror_r F
+GLIBC_2.25 __strfmon_l F
+GLIBC_2.25 __strftime_l F
+GLIBC_2.25 __strncasecmp_l F
+GLIBC_2.25 __strncat_chk F
+GLIBC_2.25 __strncpy_chk F
+GLIBC_2.25 __strndup F
+GLIBC_2.25 __strsep_g F
+GLIBC_2.25 __strtod_internal F
+GLIBC_2.25 __strtod_l F
+GLIBC_2.25 __strtof_internal F
+GLIBC_2.25 __strtof_l F
+GLIBC_2.25 __strtok_r F
+GLIBC_2.25 __strtol_internal F
+GLIBC_2.25 __strtol_l F
+GLIBC_2.25 __strtold_internal F
+GLIBC_2.25 __strtold_l F
+GLIBC_2.25 __strtoll_internal F
+GLIBC_2.25 __strtoll_l F
+GLIBC_2.25 __strtoul_internal F
+GLIBC_2.25 __strtoul_l F
+GLIBC_2.25 __strtoull_internal F
+GLIBC_2.25 __strtoull_l F
+GLIBC_2.25 __strverscmp F
+GLIBC_2.25 __strxfrm_l F
+GLIBC_2.25 __swprintf_chk F
+GLIBC_2.25 __sysconf F
+GLIBC_2.25 __syslog_chk F
+GLIBC_2.25 __sysv_signal F
+GLIBC_2.25 __timezone D 0x4
+GLIBC_2.25 __toascii_l F
+GLIBC_2.25 __tolower_l F
+GLIBC_2.25 __toupper_l F
+GLIBC_2.25 __towctrans F
+GLIBC_2.25 __towctrans_l F
+GLIBC_2.25 __towlower_l F
+GLIBC_2.25 __towupper_l F
+GLIBC_2.25 __ttyname_r_chk F
+GLIBC_2.25 __tzname D 0x8
+GLIBC_2.25 __uflow F
+GLIBC_2.25 __underflow F
+GLIBC_2.25 __uselocale F
+GLIBC_2.25 __vasprintf_chk F
+GLIBC_2.25 __vdprintf_chk F
+GLIBC_2.25 __vfork F
+GLIBC_2.25 __vfprintf_chk F
+GLIBC_2.25 __vfscanf F
+GLIBC_2.25 __vfwprintf_chk F
+GLIBC_2.25 __vprintf_chk F
+GLIBC_2.25 __vsnprintf F
+GLIBC_2.25 __vsnprintf_chk F
+GLIBC_2.25 __vsprintf_chk F
+GLIBC_2.25 __vsscanf F
+GLIBC_2.25 __vswprintf_chk F
+GLIBC_2.25 __vsyslog_chk F
+GLIBC_2.25 __vwprintf_chk F
+GLIBC_2.25 __wait F
+GLIBC_2.25 __waitpid F
+GLIBC_2.25 __wcpcpy_chk F
+GLIBC_2.25 __wcpncpy_chk F
+GLIBC_2.25 __wcrtomb_chk F
+GLIBC_2.25 __wcscasecmp_l F
+GLIBC_2.25 __wcscat_chk F
+GLIBC_2.25 __wcscoll_l F
+GLIBC_2.25 __wcscpy_chk F
+GLIBC_2.25 __wcsftime_l F
+GLIBC_2.25 __wcsncasecmp_l F
+GLIBC_2.25 __wcsncat_chk F
+GLIBC_2.25 __wcsncpy_chk F
+GLIBC_2.25 __wcsnrtombs_chk F
+GLIBC_2.25 __wcsrtombs_chk F
+GLIBC_2.25 __wcstod_internal F
+GLIBC_2.25 __wcstod_l F
+GLIBC_2.25 __wcstof_internal F
+GLIBC_2.25 __wcstof_l F
+GLIBC_2.25 __wcstol_internal F
+GLIBC_2.25 __wcstol_l F
+GLIBC_2.25 __wcstold_internal F
+GLIBC_2.25 __wcstold_l F
+GLIBC_2.25 __wcstoll_internal F
+GLIBC_2.25 __wcstoll_l F
+GLIBC_2.25 __wcstombs_chk F
+GLIBC_2.25 __wcstoul_internal F
+GLIBC_2.25 __wcstoul_l F
+GLIBC_2.25 __wcstoull_internal F
+GLIBC_2.25 __wcstoull_l F
+GLIBC_2.25 __wcsxfrm_l F
+GLIBC_2.25 __wctomb_chk F
+GLIBC_2.25 __wctrans_l F
+GLIBC_2.25 __wctype_l F
+GLIBC_2.25 __wmemcpy_chk F
+GLIBC_2.25 __wmemmove_chk F
+GLIBC_2.25 __wmempcpy_chk F
+GLIBC_2.25 __wmemset_chk F
+GLIBC_2.25 __woverflow F
+GLIBC_2.25 __wprintf_chk F
+GLIBC_2.25 __write F
+GLIBC_2.25 __wuflow F
+GLIBC_2.25 __wunderflow F
+GLIBC_2.25 __xmknod F
+GLIBC_2.25 __xmknodat F
+GLIBC_2.25 __xpg_basename F
+GLIBC_2.25 __xpg_sigpause F
+GLIBC_2.25 __xpg_strerror_r F
+GLIBC_2.25 __xstat F
+GLIBC_2.25 __xstat64 F
+GLIBC_2.25 _authenticate F
+GLIBC_2.25 _dl_mcount_wrapper F
+GLIBC_2.25 _dl_mcount_wrapper_check F
+GLIBC_2.25 _environ D 0x4
+GLIBC_2.25 _exit F
+GLIBC_2.25 _flushlbf F
+GLIBC_2.25 _libc_intl_domainname D 0x5
+GLIBC_2.25 _longjmp F
+GLIBC_2.25 _mcleanup F
+GLIBC_2.25 _mcount F
+GLIBC_2.25 _nl_default_dirname D 0x12
+GLIBC_2.25 _nl_domain_bindings D 0x4
+GLIBC_2.25 _nl_msg_cat_cntr D 0x4
+GLIBC_2.25 _null_auth D 0xc
+GLIBC_2.25 _obstack_allocated_p F
+GLIBC_2.25 _obstack_begin F
+GLIBC_2.25 _obstack_begin_1 F
+GLIBC_2.25 _obstack_free F
+GLIBC_2.25 _obstack_memory_used F
+GLIBC_2.25 _obstack_newchunk F
+GLIBC_2.25 _res D 0x200
+GLIBC_2.25 _res_hconf D 0x30
+GLIBC_2.25 _rpc_dtablesize F
+GLIBC_2.25 _seterr_reply F
+GLIBC_2.25 _setjmp F
+GLIBC_2.25 _sys_errlist D 0x21c
+GLIBC_2.25 _sys_nerr D 0x4
+GLIBC_2.25 _sys_siglist D 0x104
+GLIBC_2.25 _tolower F
+GLIBC_2.25 _toupper F
+GLIBC_2.25 a64l F
+GLIBC_2.25 abort F
+GLIBC_2.25 abs F
+GLIBC_2.25 accept F
+GLIBC_2.25 accept4 F
+GLIBC_2.25 access F
+GLIBC_2.25 acct F
+GLIBC_2.25 addmntent F
+GLIBC_2.25 addseverity F
+GLIBC_2.25 adjtime F
+GLIBC_2.25 adjtimex F
+GLIBC_2.25 alarm F
+GLIBC_2.25 aligned_alloc F
+GLIBC_2.25 alphasort F
+GLIBC_2.25 alphasort64 F
+GLIBC_2.25 argp_err_exit_status D 0x4
+GLIBC_2.25 argp_error F
+GLIBC_2.25 argp_failure F
+GLIBC_2.25 argp_help F
+GLIBC_2.25 argp_parse F
+GLIBC_2.25 argp_program_bug_address D 0x4
+GLIBC_2.25 argp_program_version D 0x4
+GLIBC_2.25 argp_program_version_hook D 0x4
+GLIBC_2.25 argp_state_help F
+GLIBC_2.25 argp_usage F
+GLIBC_2.25 argz_add F
+GLIBC_2.25 argz_add_sep F
+GLIBC_2.25 argz_append F
+GLIBC_2.25 argz_count F
+GLIBC_2.25 argz_create F
+GLIBC_2.25 argz_create_sep F
+GLIBC_2.25 argz_delete F
+GLIBC_2.25 argz_extract F
+GLIBC_2.25 argz_insert F
+GLIBC_2.25 argz_next F
+GLIBC_2.25 argz_replace F
+GLIBC_2.25 argz_stringify F
+GLIBC_2.25 asctime F
+GLIBC_2.25 asctime_r F
+GLIBC_2.25 asprintf F
+GLIBC_2.25 atof F
+GLIBC_2.25 atoi F
+GLIBC_2.25 atol F
+GLIBC_2.25 atoll F
+GLIBC_2.25 authdes_create F
+GLIBC_2.25 authdes_getucred F
+GLIBC_2.25 authdes_pk_create F
+GLIBC_2.25 authnone_create F
+GLIBC_2.25 authunix_create F
+GLIBC_2.25 authunix_create_default F
+GLIBC_2.25 backtrace F
+GLIBC_2.25 backtrace_symbols F
+GLIBC_2.25 backtrace_symbols_fd F
+GLIBC_2.25 basename F
+GLIBC_2.25 bcmp F
+GLIBC_2.25 bcopy F
+GLIBC_2.25 bind F
+GLIBC_2.25 bind_textdomain_codeset F
+GLIBC_2.25 bindresvport F
+GLIBC_2.25 bindtextdomain F
+GLIBC_2.25 brk F
+GLIBC_2.25 bsd_signal F
+GLIBC_2.25 bsearch F
+GLIBC_2.25 btowc F
+GLIBC_2.25 bzero F
+GLIBC_2.25 c16rtomb F
+GLIBC_2.25 c32rtomb F
+GLIBC_2.25 calloc F
+GLIBC_2.25 callrpc F
+GLIBC_2.25 canonicalize_file_name F
+GLIBC_2.25 capget F
+GLIBC_2.25 capset F
+GLIBC_2.25 catclose F
+GLIBC_2.25 catgets F
+GLIBC_2.25 catopen F
+GLIBC_2.25 cbc_crypt F
+GLIBC_2.25 cfgetispeed F
+GLIBC_2.25 cfgetospeed F
+GLIBC_2.25 cfmakeraw F
+GLIBC_2.25 cfree F
+GLIBC_2.25 cfsetispeed F
+GLIBC_2.25 cfsetospeed F
+GLIBC_2.25 cfsetspeed F
+GLIBC_2.25 chdir F
+GLIBC_2.25 chflags F
+GLIBC_2.25 chmod F
+GLIBC_2.25 chown F
+GLIBC_2.25 chroot F
+GLIBC_2.25 clearenv F
+GLIBC_2.25 clearerr F
+GLIBC_2.25 clearerr_unlocked F
+GLIBC_2.25 clnt_broadcast F
+GLIBC_2.25 clnt_create F
+GLIBC_2.25 clnt_pcreateerror F
+GLIBC_2.25 clnt_perrno F
+GLIBC_2.25 clnt_perror F
+GLIBC_2.25 clnt_spcreateerror F
+GLIBC_2.25 clnt_sperrno F
+GLIBC_2.25 clnt_sperror F
+GLIBC_2.25 clntraw_create F
+GLIBC_2.25 clnttcp_create F
+GLIBC_2.25 clntudp_bufcreate F
+GLIBC_2.25 clntudp_create F
+GLIBC_2.25 clntunix_create F
+GLIBC_2.25 clock F
+GLIBC_2.25 clock_adjtime F
+GLIBC_2.25 clock_getcpuclockid F
+GLIBC_2.25 clock_getres F
+GLIBC_2.25 clock_gettime F
+GLIBC_2.25 clock_nanosleep F
+GLIBC_2.25 clock_settime F
+GLIBC_2.25 clone F
+GLIBC_2.25 close F
+GLIBC_2.25 closedir F
+GLIBC_2.25 closelog F
+GLIBC_2.25 confstr F
+GLIBC_2.25 connect F
+GLIBC_2.25 copysign F
+GLIBC_2.25 copysignf F
+GLIBC_2.25 copysignl F
+GLIBC_2.25 creat F
+GLIBC_2.25 creat64 F
+GLIBC_2.25 ctermid F
+GLIBC_2.25 ctime F
+GLIBC_2.25 ctime_r F
+GLIBC_2.25 cuserid F
+GLIBC_2.25 daemon F
+GLIBC_2.25 daylight D 0x4
+GLIBC_2.25 dcgettext F
+GLIBC_2.25 dcngettext F
+GLIBC_2.25 delete_module F
+GLIBC_2.25 des_setparity F
+GLIBC_2.25 dgettext F
+GLIBC_2.25 difftime F
+GLIBC_2.25 dirfd F
+GLIBC_2.25 dirname F
+GLIBC_2.25 div F
+GLIBC_2.25 dl_iterate_phdr F
+GLIBC_2.25 dngettext F
+GLIBC_2.25 dprintf F
+GLIBC_2.25 drand48 F
+GLIBC_2.25 drand48_r F
+GLIBC_2.25 dup F
+GLIBC_2.25 dup2 F
+GLIBC_2.25 dup3 F
+GLIBC_2.25 duplocale F
+GLIBC_2.25 dysize F
+GLIBC_2.25 eaccess F
+GLIBC_2.25 ecb_crypt F
+GLIBC_2.25 ecvt F
+GLIBC_2.25 ecvt_r F
+GLIBC_2.25 endaliasent F
+GLIBC_2.25 endfsent F
+GLIBC_2.25 endgrent F
+GLIBC_2.25 endhostent F
+GLIBC_2.25 endmntent F
+GLIBC_2.25 endnetent F
+GLIBC_2.25 endnetgrent F
+GLIBC_2.25 endprotoent F
+GLIBC_2.25 endpwent F
+GLIBC_2.25 endrpcent F
+GLIBC_2.25 endservent F
+GLIBC_2.25 endsgent F
+GLIBC_2.25 endspent F
+GLIBC_2.25 endttyent F
+GLIBC_2.25 endusershell F
+GLIBC_2.25 endutent F
+GLIBC_2.25 endutxent F
+GLIBC_2.25 environ D 0x4
+GLIBC_2.25 envz_add F
+GLIBC_2.25 envz_entry F
+GLIBC_2.25 envz_get F
+GLIBC_2.25 envz_merge F
+GLIBC_2.25 envz_remove F
+GLIBC_2.25 envz_strip F
+GLIBC_2.25 epoll_create F
+GLIBC_2.25 epoll_create1 F
+GLIBC_2.25 epoll_ctl F
+GLIBC_2.25 epoll_pwait F
+GLIBC_2.25 epoll_wait F
+GLIBC_2.25 erand48 F
+GLIBC_2.25 erand48_r F
+GLIBC_2.25 err F
+GLIBC_2.25 error F
+GLIBC_2.25 error_at_line F
+GLIBC_2.25 error_message_count D 0x4
+GLIBC_2.25 error_one_per_line D 0x4
+GLIBC_2.25 error_print_progname D 0x4
+GLIBC_2.25 errx F
+GLIBC_2.25 ether_aton F
+GLIBC_2.25 ether_aton_r F
+GLIBC_2.25 ether_hostton F
+GLIBC_2.25 ether_line F
+GLIBC_2.25 ether_ntoa F
+GLIBC_2.25 ether_ntoa_r F
+GLIBC_2.25 ether_ntohost F
+GLIBC_2.25 euidaccess F
+GLIBC_2.25 eventfd F
+GLIBC_2.25 eventfd_read F
+GLIBC_2.25 eventfd_write F
+GLIBC_2.25 execl F
+GLIBC_2.25 execle F
+GLIBC_2.25 execlp F
+GLIBC_2.25 execv F
+GLIBC_2.25 execve F
+GLIBC_2.25 execvp F
+GLIBC_2.25 execvpe F
+GLIBC_2.25 exit F
+GLIBC_2.25 explicit_bzero F
+GLIBC_2.25 faccessat F
+GLIBC_2.25 fallocate F
+GLIBC_2.25 fallocate64 F
+GLIBC_2.25 fanotify_init F
+GLIBC_2.25 fanotify_mark F
+GLIBC_2.25 fattach F
+GLIBC_2.25 fchdir F
+GLIBC_2.25 fchflags F
+GLIBC_2.25 fchmod F
+GLIBC_2.25 fchmodat F
+GLIBC_2.25 fchown F
+GLIBC_2.25 fchownat F
+GLIBC_2.25 fclose F
+GLIBC_2.25 fcloseall F
+GLIBC_2.25 fcntl F
+GLIBC_2.25 fcvt F
+GLIBC_2.25 fcvt_r F
+GLIBC_2.25 fdatasync F
+GLIBC_2.25 fdetach F
+GLIBC_2.25 fdopen F
+GLIBC_2.25 fdopendir F
+GLIBC_2.25 feof F
+GLIBC_2.25 feof_unlocked F
+GLIBC_2.25 ferror F
+GLIBC_2.25 ferror_unlocked F
+GLIBC_2.25 fexecve F
+GLIBC_2.25 fflush F
+GLIBC_2.25 fflush_unlocked F
+GLIBC_2.25 ffs F
+GLIBC_2.25 ffsl F
+GLIBC_2.25 ffsll F
+GLIBC_2.25 fgetc F
+GLIBC_2.25 fgetc_unlocked F
+GLIBC_2.25 fgetgrent F
+GLIBC_2.25 fgetgrent_r F
+GLIBC_2.25 fgetpos F
+GLIBC_2.25 fgetpos64 F
+GLIBC_2.25 fgetpwent F
+GLIBC_2.25 fgetpwent_r F
+GLIBC_2.25 fgets F
+GLIBC_2.25 fgets_unlocked F
+GLIBC_2.25 fgetsgent F
+GLIBC_2.25 fgetsgent_r F
+GLIBC_2.25 fgetspent F
+GLIBC_2.25 fgetspent_r F
+GLIBC_2.25 fgetwc F
+GLIBC_2.25 fgetwc_unlocked F
+GLIBC_2.25 fgetws F
+GLIBC_2.25 fgetws_unlocked F
+GLIBC_2.25 fgetxattr F
+GLIBC_2.25 fileno F
+GLIBC_2.25 fileno_unlocked F
+GLIBC_2.25 finite F
+GLIBC_2.25 finitef F
+GLIBC_2.25 finitel F
+GLIBC_2.25 flistxattr F
+GLIBC_2.25 flock F
+GLIBC_2.25 flockfile F
+GLIBC_2.25 fmemopen F
+GLIBC_2.25 fmtmsg F
+GLIBC_2.25 fnmatch F
+GLIBC_2.25 fopen F
+GLIBC_2.25 fopen64 F
+GLIBC_2.25 fopencookie F
+GLIBC_2.25 fork F
+GLIBC_2.25 fpathconf F
+GLIBC_2.25 fprintf F
+GLIBC_2.25 fputc F
+GLIBC_2.25 fputc_unlocked F
+GLIBC_2.25 fputs F
+GLIBC_2.25 fputs_unlocked F
+GLIBC_2.25 fputwc F
+GLIBC_2.25 fputwc_unlocked F
+GLIBC_2.25 fputws F
+GLIBC_2.25 fputws_unlocked F
+GLIBC_2.25 fread F
+GLIBC_2.25 fread_unlocked F
+GLIBC_2.25 free F
+GLIBC_2.25 freeaddrinfo F
+GLIBC_2.25 freeifaddrs F
+GLIBC_2.25 freelocale F
+GLIBC_2.25 fremovexattr F
+GLIBC_2.25 freopen F
+GLIBC_2.25 freopen64 F
+GLIBC_2.25 frexp F
+GLIBC_2.25 frexpf F
+GLIBC_2.25 frexpl F
+GLIBC_2.25 fscanf F
+GLIBC_2.25 fseek F
+GLIBC_2.25 fseeko F
+GLIBC_2.25 fseeko64 F
+GLIBC_2.25 fsetpos F
+GLIBC_2.25 fsetpos64 F
+GLIBC_2.25 fsetxattr F
+GLIBC_2.25 fstatfs F
+GLIBC_2.25 fstatfs64 F
+GLIBC_2.25 fstatvfs F
+GLIBC_2.25 fstatvfs64 F
+GLIBC_2.25 fsync F
+GLIBC_2.25 ftell F
+GLIBC_2.25 ftello F
+GLIBC_2.25 ftello64 F
+GLIBC_2.25 ftime F
+GLIBC_2.25 ftok F
+GLIBC_2.25 ftruncate F
+GLIBC_2.25 ftruncate64 F
+GLIBC_2.25 ftrylockfile F
+GLIBC_2.25 fts64_children F
+GLIBC_2.25 fts64_close F
+GLIBC_2.25 fts64_open F
+GLIBC_2.25 fts64_read F
+GLIBC_2.25 fts64_set F
+GLIBC_2.25 fts_children F
+GLIBC_2.25 fts_close F
+GLIBC_2.25 fts_open F
+GLIBC_2.25 fts_read F
+GLIBC_2.25 fts_set F
+GLIBC_2.25 ftw F
+GLIBC_2.25 ftw64 F
+GLIBC_2.25 funlockfile F
+GLIBC_2.25 futimens F
+GLIBC_2.25 futimes F
+GLIBC_2.25 futimesat F
+GLIBC_2.25 fwide F
+GLIBC_2.25 fwprintf F
+GLIBC_2.25 fwrite F
+GLIBC_2.25 fwrite_unlocked F
+GLIBC_2.25 fwscanf F
+GLIBC_2.25 gai_strerror F
+GLIBC_2.25 gcvt F
+GLIBC_2.25 get_avphys_pages F
+GLIBC_2.25 get_current_dir_name F
+GLIBC_2.25 get_myaddress F
+GLIBC_2.25 get_nprocs F
+GLIBC_2.25 get_nprocs_conf F
+GLIBC_2.25 get_phys_pages F
+GLIBC_2.25 getaddrinfo F
+GLIBC_2.25 getaliasbyname F
+GLIBC_2.25 getaliasbyname_r F
+GLIBC_2.25 getaliasent F
+GLIBC_2.25 getaliasent_r F
+GLIBC_2.25 getauxval F
+GLIBC_2.25 getc F
+GLIBC_2.25 getc_unlocked F
+GLIBC_2.25 getchar F
+GLIBC_2.25 getchar_unlocked F
+GLIBC_2.25 getcontext F
+GLIBC_2.25 getcwd F
+GLIBC_2.25 getdate F
+GLIBC_2.25 getdate_err D 0x4
+GLIBC_2.25 getdate_r F
+GLIBC_2.25 getdelim F
+GLIBC_2.25 getdirentries F
+GLIBC_2.25 getdirentries64 F
+GLIBC_2.25 getdomainname F
+GLIBC_2.25 getdtablesize F
+GLIBC_2.25 getegid F
+GLIBC_2.25 getentropy F
+GLIBC_2.25 getenv F
+GLIBC_2.25 geteuid F
+GLIBC_2.25 getfsent F
+GLIBC_2.25 getfsfile F
+GLIBC_2.25 getfsspec F
+GLIBC_2.25 getgid F
+GLIBC_2.25 getgrent F
+GLIBC_2.25 getgrent_r F
+GLIBC_2.25 getgrgid F
+GLIBC_2.25 getgrgid_r F
+GLIBC_2.25 getgrnam F
+GLIBC_2.25 getgrnam_r F
+GLIBC_2.25 getgrouplist F
+GLIBC_2.25 getgroups F
+GLIBC_2.25 gethostbyaddr F
+GLIBC_2.25 gethostbyaddr_r F
+GLIBC_2.25 gethostbyname F
+GLIBC_2.25 gethostbyname2 F
+GLIBC_2.25 gethostbyname2_r F
+GLIBC_2.25 gethostbyname_r F
+GLIBC_2.25 gethostent F
+GLIBC_2.25 gethostent_r F
+GLIBC_2.25 gethostid F
+GLIBC_2.25 gethostname F
+GLIBC_2.25 getifaddrs F
+GLIBC_2.25 getipv4sourcefilter F
+GLIBC_2.25 getitimer F
+GLIBC_2.25 getline F
+GLIBC_2.25 getloadavg F
+GLIBC_2.25 getlogin F
+GLIBC_2.25 getlogin_r F
+GLIBC_2.25 getmntent F
+GLIBC_2.25 getmntent_r F
+GLIBC_2.25 getmsg F
+GLIBC_2.25 getnameinfo F
+GLIBC_2.25 getnetbyaddr F
+GLIBC_2.25 getnetbyaddr_r F
+GLIBC_2.25 getnetbyname F
+GLIBC_2.25 getnetbyname_r F
+GLIBC_2.25 getnetent F
+GLIBC_2.25 getnetent_r F
+GLIBC_2.25 getnetgrent F
+GLIBC_2.25 getnetgrent_r F
+GLIBC_2.25 getnetname F
+GLIBC_2.25 getopt F
+GLIBC_2.25 getopt_long F
+GLIBC_2.25 getopt_long_only F
+GLIBC_2.25 getpagesize F
+GLIBC_2.25 getpass F
+GLIBC_2.25 getpeername F
+GLIBC_2.25 getpgid F
+GLIBC_2.25 getpgrp F
+GLIBC_2.25 getpid F
+GLIBC_2.25 getpmsg F
+GLIBC_2.25 getppid F
+GLIBC_2.25 getpriority F
+GLIBC_2.25 getprotobyname F
+GLIBC_2.25 getprotobyname_r F
+GLIBC_2.25 getprotobynumber F
+GLIBC_2.25 getprotobynumber_r F
+GLIBC_2.25 getprotoent F
+GLIBC_2.25 getprotoent_r F
+GLIBC_2.25 getpt F
+GLIBC_2.25 getpublickey F
+GLIBC_2.25 getpw F
+GLIBC_2.25 getpwent F
+GLIBC_2.25 getpwent_r F
+GLIBC_2.25 getpwnam F
+GLIBC_2.25 getpwnam_r F
+GLIBC_2.25 getpwuid F
+GLIBC_2.25 getpwuid_r F
+GLIBC_2.25 getrandom F
+GLIBC_2.25 getresgid F
+GLIBC_2.25 getresuid F
+GLIBC_2.25 getrlimit F
+GLIBC_2.25 getrlimit64 F
+GLIBC_2.25 getrpcbyname F
+GLIBC_2.25 getrpcbyname_r F
+GLIBC_2.25 getrpcbynumber F
+GLIBC_2.25 getrpcbynumber_r F
+GLIBC_2.25 getrpcent F
+GLIBC_2.25 getrpcent_r F
+GLIBC_2.25 getrpcport F
+GLIBC_2.25 getrusage F
+GLIBC_2.25 gets F
+GLIBC_2.25 getsecretkey F
+GLIBC_2.25 getservbyname F
+GLIBC_2.25 getservbyname_r F
+GLIBC_2.25 getservbyport F
+GLIBC_2.25 getservbyport_r F
+GLIBC_2.25 getservent F
+GLIBC_2.25 getservent_r F
+GLIBC_2.25 getsgent F
+GLIBC_2.25 getsgent_r F
+GLIBC_2.25 getsgnam F
+GLIBC_2.25 getsgnam_r F
+GLIBC_2.25 getsid F
+GLIBC_2.25 getsockname F
+GLIBC_2.25 getsockopt F
+GLIBC_2.25 getsourcefilter F
+GLIBC_2.25 getspent F
+GLIBC_2.25 getspent_r F
+GLIBC_2.25 getspnam F
+GLIBC_2.25 getspnam_r F
+GLIBC_2.25 getsubopt F
+GLIBC_2.25 gettext F
+GLIBC_2.25 gettimeofday F
+GLIBC_2.25 getttyent F
+GLIBC_2.25 getttynam F
+GLIBC_2.25 getuid F
+GLIBC_2.25 getusershell F
+GLIBC_2.25 getutent F
+GLIBC_2.25 getutent_r F
+GLIBC_2.25 getutid F
+GLIBC_2.25 getutid_r F
+GLIBC_2.25 getutline F
+GLIBC_2.25 getutline_r F
+GLIBC_2.25 getutmp F
+GLIBC_2.25 getutmpx F
+GLIBC_2.25 getutxent F
+GLIBC_2.25 getutxid F
+GLIBC_2.25 getutxline F
+GLIBC_2.25 getw F
+GLIBC_2.25 getwc F
+GLIBC_2.25 getwc_unlocked F
+GLIBC_2.25 getwchar F
+GLIBC_2.25 getwchar_unlocked F
+GLIBC_2.25 getwd F
+GLIBC_2.25 getxattr F
+GLIBC_2.25 glob F
+GLIBC_2.25 glob64 F
+GLIBC_2.25 glob_pattern_p F
+GLIBC_2.25 globfree F
+GLIBC_2.25 globfree64 F
+GLIBC_2.25 gmtime F
+GLIBC_2.25 gmtime_r F
+GLIBC_2.25 gnu_dev_major F
+GLIBC_2.25 gnu_dev_makedev F
+GLIBC_2.25 gnu_dev_minor F
+GLIBC_2.25 gnu_get_libc_release F
+GLIBC_2.25 gnu_get_libc_version F
+GLIBC_2.25 grantpt F
+GLIBC_2.25 group_member F
+GLIBC_2.25 gsignal F
+GLIBC_2.25 gtty F
+GLIBC_2.25 h_errlist D 0x14
+GLIBC_2.25 h_nerr D 0x4
+GLIBC_2.25 hasmntopt F
+GLIBC_2.25 hcreate F
+GLIBC_2.25 hcreate_r F
+GLIBC_2.25 hdestroy F
+GLIBC_2.25 hdestroy_r F
+GLIBC_2.25 herror F
+GLIBC_2.25 host2netname F
+GLIBC_2.25 hsearch F
+GLIBC_2.25 hsearch_r F
+GLIBC_2.25 hstrerror F
+GLIBC_2.25 htonl F
+GLIBC_2.25 htons F
+GLIBC_2.25 iconv F
+GLIBC_2.25 iconv_close F
+GLIBC_2.25 iconv_open F
+GLIBC_2.25 if_freenameindex F
+GLIBC_2.25 if_indextoname F
+GLIBC_2.25 if_nameindex F
+GLIBC_2.25 if_nametoindex F
+GLIBC_2.25 imaxabs F
+GLIBC_2.25 imaxdiv F
+GLIBC_2.25 in6addr_any D 0x10
+GLIBC_2.25 in6addr_loopback D 0x10
+GLIBC_2.25 index F
+GLIBC_2.25 inet6_opt_append F
+GLIBC_2.25 inet6_opt_find F
+GLIBC_2.25 inet6_opt_finish F
+GLIBC_2.25 inet6_opt_get_val F
+GLIBC_2.25 inet6_opt_init F
+GLIBC_2.25 inet6_opt_next F
+GLIBC_2.25 inet6_opt_set_val F
+GLIBC_2.25 inet6_option_alloc F
+GLIBC_2.25 inet6_option_append F
+GLIBC_2.25 inet6_option_find F
+GLIBC_2.25 inet6_option_init F
+GLIBC_2.25 inet6_option_next F
+GLIBC_2.25 inet6_option_space F
+GLIBC_2.25 inet6_rth_add F
+GLIBC_2.25 inet6_rth_getaddr F
+GLIBC_2.25 inet6_rth_init F
+GLIBC_2.25 inet6_rth_reverse F
+GLIBC_2.25 inet6_rth_segments F
+GLIBC_2.25 inet6_rth_space F
+GLIBC_2.25 inet_addr F
+GLIBC_2.25 inet_aton F
+GLIBC_2.25 inet_lnaof F
+GLIBC_2.25 inet_makeaddr F
+GLIBC_2.25 inet_netof F
+GLIBC_2.25 inet_network F
+GLIBC_2.25 inet_nsap_addr F
+GLIBC_2.25 inet_nsap_ntoa F
+GLIBC_2.25 inet_ntoa F
+GLIBC_2.25 inet_ntop F
+GLIBC_2.25 inet_pton F
+GLIBC_2.25 init_module F
+GLIBC_2.25 initgroups F
+GLIBC_2.25 initstate F
+GLIBC_2.25 initstate_r F
+GLIBC_2.25 innetgr F
+GLIBC_2.25 inotify_add_watch F
+GLIBC_2.25 inotify_init F
+GLIBC_2.25 inotify_init1 F
+GLIBC_2.25 inotify_rm_watch F
+GLIBC_2.25 insque F
+GLIBC_2.25 ioctl F
+GLIBC_2.25 iruserok F
+GLIBC_2.25 iruserok_af F
+GLIBC_2.25 isalnum F
+GLIBC_2.25 isalnum_l F
+GLIBC_2.25 isalpha F
+GLIBC_2.25 isalpha_l F
+GLIBC_2.25 isascii F
+GLIBC_2.25 isastream F
+GLIBC_2.25 isatty F
+GLIBC_2.25 isblank F
+GLIBC_2.25 isblank_l F
+GLIBC_2.25 iscntrl F
+GLIBC_2.25 iscntrl_l F
+GLIBC_2.25 isctype F
+GLIBC_2.25 isdigit F
+GLIBC_2.25 isdigit_l F
+GLIBC_2.25 isfdtype F
+GLIBC_2.25 isgraph F
+GLIBC_2.25 isgraph_l F
+GLIBC_2.25 isinf F
+GLIBC_2.25 isinff F
+GLIBC_2.25 isinfl F
+GLIBC_2.25 islower F
+GLIBC_2.25 islower_l F
+GLIBC_2.25 isnan F
+GLIBC_2.25 isnanf F
+GLIBC_2.25 isnanl F
+GLIBC_2.25 isprint F
+GLIBC_2.25 isprint_l F
+GLIBC_2.25 ispunct F
+GLIBC_2.25 ispunct_l F
+GLIBC_2.25 isspace F
+GLIBC_2.25 isspace_l F
+GLIBC_2.25 isupper F
+GLIBC_2.25 isupper_l F
+GLIBC_2.25 iswalnum F
+GLIBC_2.25 iswalnum_l F
+GLIBC_2.25 iswalpha F
+GLIBC_2.25 iswalpha_l F
+GLIBC_2.25 iswblank F
+GLIBC_2.25 iswblank_l F
+GLIBC_2.25 iswcntrl F
+GLIBC_2.25 iswcntrl_l F
+GLIBC_2.25 iswctype F
+GLIBC_2.25 iswctype_l F
+GLIBC_2.25 iswdigit F
+GLIBC_2.25 iswdigit_l F
+GLIBC_2.25 iswgraph F
+GLIBC_2.25 iswgraph_l F
+GLIBC_2.25 iswlower F
+GLIBC_2.25 iswlower_l F
+GLIBC_2.25 iswprint F
+GLIBC_2.25 iswprint_l F
+GLIBC_2.25 iswpunct F
+GLIBC_2.25 iswpunct_l F
+GLIBC_2.25 iswspace F
+GLIBC_2.25 iswspace_l F
+GLIBC_2.25 iswupper F
+GLIBC_2.25 iswupper_l F
+GLIBC_2.25 iswxdigit F
+GLIBC_2.25 iswxdigit_l F
+GLIBC_2.25 isxdigit F
+GLIBC_2.25 isxdigit_l F
+GLIBC_2.25 jrand48 F
+GLIBC_2.25 jrand48_r F
+GLIBC_2.25 key_decryptsession F
+GLIBC_2.25 key_decryptsession_pk F
+GLIBC_2.25 key_encryptsession F
+GLIBC_2.25 key_encryptsession_pk F
+GLIBC_2.25 key_gendes F
+GLIBC_2.25 key_get_conv F
+GLIBC_2.25 key_secretkey_is_set F
+GLIBC_2.25 key_setnet F
+GLIBC_2.25 key_setsecret F
+GLIBC_2.25 kill F
+GLIBC_2.25 killpg F
+GLIBC_2.25 klogctl F
+GLIBC_2.25 l64a F
+GLIBC_2.25 labs F
+GLIBC_2.25 lchmod F
+GLIBC_2.25 lchown F
+GLIBC_2.25 lckpwdf F
+GLIBC_2.25 lcong48 F
+GLIBC_2.25 lcong48_r F
+GLIBC_2.25 ldexp F
+GLIBC_2.25 ldexpf F
+GLIBC_2.25 ldexpl F
+GLIBC_2.25 ldiv F
+GLIBC_2.25 lfind F
+GLIBC_2.25 lgetxattr F
+GLIBC_2.25 link F
+GLIBC_2.25 linkat F
+GLIBC_2.25 listen F
+GLIBC_2.25 listxattr F
+GLIBC_2.25 llabs F
+GLIBC_2.25 lldiv F
+GLIBC_2.25 llistxattr F
+GLIBC_2.25 llseek F
+GLIBC_2.25 localeconv F
+GLIBC_2.25 localtime F
+GLIBC_2.25 localtime_r F
+GLIBC_2.25 lockf F
+GLIBC_2.25 lockf64 F
+GLIBC_2.25 longjmp F
+GLIBC_2.25 lrand48 F
+GLIBC_2.25 lrand48_r F
+GLIBC_2.25 lremovexattr F
+GLIBC_2.25 lsearch F
+GLIBC_2.25 lseek F
+GLIBC_2.25 lseek64 F
+GLIBC_2.25 lsetxattr F
+GLIBC_2.25 lutimes F
+GLIBC_2.25 madvise F
+GLIBC_2.25 makecontext F
+GLIBC_2.25 mallinfo F
+GLIBC_2.25 malloc F
+GLIBC_2.25 malloc_info F
+GLIBC_2.25 malloc_stats F
+GLIBC_2.25 malloc_trim F
+GLIBC_2.25 malloc_usable_size F
+GLIBC_2.25 mallopt F
+GLIBC_2.25 mallwatch D 0x4
+GLIBC_2.25 mblen F
+GLIBC_2.25 mbrlen F
+GLIBC_2.25 mbrtoc16 F
+GLIBC_2.25 mbrtoc32 F
+GLIBC_2.25 mbrtowc F
+GLIBC_2.25 mbsinit F
+GLIBC_2.25 mbsnrtowcs F
+GLIBC_2.25 mbsrtowcs F
+GLIBC_2.25 mbstowcs F
+GLIBC_2.25 mbtowc F
+GLIBC_2.25 mcheck F
+GLIBC_2.25 mcheck_check_all F
+GLIBC_2.25 mcheck_pedantic F
+GLIBC_2.25 memalign F
+GLIBC_2.25 memccpy F
+GLIBC_2.25 memchr F
+GLIBC_2.25 memcmp F
+GLIBC_2.25 memcpy F
+GLIBC_2.25 memfrob F
+GLIBC_2.25 memmem F
+GLIBC_2.25 memmove F
+GLIBC_2.25 mempcpy F
+GLIBC_2.25 memrchr F
+GLIBC_2.25 memset F
+GLIBC_2.25 mincore F
+GLIBC_2.25 mkdir F
+GLIBC_2.25 mkdirat F
+GLIBC_2.25 mkdtemp F
+GLIBC_2.25 mkfifo F
+GLIBC_2.25 mkfifoat F
+GLIBC_2.25 mkostemp F
+GLIBC_2.25 mkostemp64 F
+GLIBC_2.25 mkostemps F
+GLIBC_2.25 mkostemps64 F
+GLIBC_2.25 mkstemp F
+GLIBC_2.25 mkstemp64 F
+GLIBC_2.25 mkstemps F
+GLIBC_2.25 mkstemps64 F
+GLIBC_2.25 mktemp F
+GLIBC_2.25 mktime F
+GLIBC_2.25 mlock F
+GLIBC_2.25 mlockall F
+GLIBC_2.25 mmap F
+GLIBC_2.25 mmap64 F
+GLIBC_2.25 modf F
+GLIBC_2.25 modff F
+GLIBC_2.25 modfl F
+GLIBC_2.25 moncontrol F
+GLIBC_2.25 monstartup F
+GLIBC_2.25 mount F
+GLIBC_2.25 mprobe F
+GLIBC_2.25 mprotect F
+GLIBC_2.25 mrand48 F
+GLIBC_2.25 mrand48_r F
+GLIBC_2.25 mremap F
+GLIBC_2.25 msgctl F
+GLIBC_2.25 msgget F
+GLIBC_2.25 msgrcv F
+GLIBC_2.25 msgsnd F
+GLIBC_2.25 msync F
+GLIBC_2.25 mtrace F
+GLIBC_2.25 munlock F
+GLIBC_2.25 munlockall F
+GLIBC_2.25 munmap F
+GLIBC_2.25 muntrace F
+GLIBC_2.25 name_to_handle_at F
+GLIBC_2.25 nanosleep F
+GLIBC_2.25 netname2host F
+GLIBC_2.25 netname2user F
+GLIBC_2.25 newlocale F
+GLIBC_2.25 nfsservctl F
+GLIBC_2.25 nftw F
+GLIBC_2.25 nftw64 F
+GLIBC_2.25 ngettext F
+GLIBC_2.25 nice F
+GLIBC_2.25 nl_langinfo F
+GLIBC_2.25 nl_langinfo_l F
+GLIBC_2.25 nrand48 F
+GLIBC_2.25 nrand48_r F
+GLIBC_2.25 ntohl F
+GLIBC_2.25 ntohs F
+GLIBC_2.25 ntp_adjtime F
+GLIBC_2.25 ntp_gettime F
+GLIBC_2.25 ntp_gettimex F
+GLIBC_2.25 obstack_alloc_failed_handler D 0x4
+GLIBC_2.25 obstack_exit_failure D 0x4
+GLIBC_2.25 obstack_free F
+GLIBC_2.25 obstack_printf F
+GLIBC_2.25 obstack_vprintf F
+GLIBC_2.25 on_exit F
+GLIBC_2.25 open F
+GLIBC_2.25 open64 F
+GLIBC_2.25 open_by_handle_at F
+GLIBC_2.25 open_memstream F
+GLIBC_2.25 open_wmemstream F
+GLIBC_2.25 openat F
+GLIBC_2.25 openat64 F
+GLIBC_2.25 opendir F
+GLIBC_2.25 openlog F
+GLIBC_2.25 optarg D 0x4
+GLIBC_2.25 opterr D 0x4
+GLIBC_2.25 optind D 0x4
+GLIBC_2.25 optopt D 0x4
+GLIBC_2.25 parse_printf_format F
+GLIBC_2.25 passwd2des F
+GLIBC_2.25 pathconf F
+GLIBC_2.25 pause F
+GLIBC_2.25 pclose F
+GLIBC_2.25 perror F
+GLIBC_2.25 personality F
+GLIBC_2.25 pipe F
+GLIBC_2.25 pipe2 F
+GLIBC_2.25 pivot_root F
+GLIBC_2.25 pmap_getmaps F
+GLIBC_2.25 pmap_getport F
+GLIBC_2.25 pmap_rmtcall F
+GLIBC_2.25 pmap_set F
+GLIBC_2.25 pmap_unset F
+GLIBC_2.25 poll F
+GLIBC_2.25 popen F
+GLIBC_2.25 posix_fadvise F
+GLIBC_2.25 posix_fadvise64 F
+GLIBC_2.25 posix_fallocate F
+GLIBC_2.25 posix_fallocate64 F
+GLIBC_2.25 posix_madvise F
+GLIBC_2.25 posix_memalign F
+GLIBC_2.25 posix_openpt F
+GLIBC_2.25 posix_spawn F
+GLIBC_2.25 posix_spawn_file_actions_addclose F
+GLIBC_2.25 posix_spawn_file_actions_adddup2 F
+GLIBC_2.25 posix_spawn_file_actions_addopen F
+GLIBC_2.25 posix_spawn_file_actions_destroy F
+GLIBC_2.25 posix_spawn_file_actions_init F
+GLIBC_2.25 posix_spawnattr_destroy F
+GLIBC_2.25 posix_spawnattr_getflags F
+GLIBC_2.25 posix_spawnattr_getpgroup F
+GLIBC_2.25 posix_spawnattr_getschedparam F
+GLIBC_2.25 posix_spawnattr_getschedpolicy F
+GLIBC_2.25 posix_spawnattr_getsigdefault F
+GLIBC_2.25 posix_spawnattr_getsigmask F
+GLIBC_2.25 posix_spawnattr_init F
+GLIBC_2.25 posix_spawnattr_setflags F
+GLIBC_2.25 posix_spawnattr_setpgroup F
+GLIBC_2.25 posix_spawnattr_setschedparam F
+GLIBC_2.25 posix_spawnattr_setschedpolicy F
+GLIBC_2.25 posix_spawnattr_setsigdefault F
+GLIBC_2.25 posix_spawnattr_setsigmask F
+GLIBC_2.25 posix_spawnp F
+GLIBC_2.25 ppoll F
+GLIBC_2.25 prctl F
+GLIBC_2.25 pread F
+GLIBC_2.25 pread64 F
+GLIBC_2.25 preadv F
+GLIBC_2.25 preadv64 F
+GLIBC_2.25 printf F
+GLIBC_2.25 printf_size F
+GLIBC_2.25 printf_size_info F
+GLIBC_2.25 prlimit F
+GLIBC_2.25 prlimit64 F
+GLIBC_2.25 process_vm_readv F
+GLIBC_2.25 process_vm_writev F
+GLIBC_2.25 profil F
+GLIBC_2.25 program_invocation_name D 0x4
+GLIBC_2.25 program_invocation_short_name D 0x4
+GLIBC_2.25 pselect F
+GLIBC_2.25 psiginfo F
+GLIBC_2.25 psignal F
+GLIBC_2.25 pthread_attr_destroy F
+GLIBC_2.25 pthread_attr_getdetachstate F
+GLIBC_2.25 pthread_attr_getinheritsched F
+GLIBC_2.25 pthread_attr_getschedparam F
+GLIBC_2.25 pthread_attr_getschedpolicy F
+GLIBC_2.25 pthread_attr_getscope F
+GLIBC_2.25 pthread_attr_init F
+GLIBC_2.25 pthread_attr_setdetachstate F
+GLIBC_2.25 pthread_attr_setinheritsched F
+GLIBC_2.25 pthread_attr_setschedparam F
+GLIBC_2.25 pthread_attr_setschedpolicy F
+GLIBC_2.25 pthread_attr_setscope F
+GLIBC_2.25 pthread_cond_broadcast F
+GLIBC_2.25 pthread_cond_destroy F
+GLIBC_2.25 pthread_cond_init F
+GLIBC_2.25 pthread_cond_signal F
+GLIBC_2.25 pthread_cond_timedwait F
+GLIBC_2.25 pthread_cond_wait F
+GLIBC_2.25 pthread_condattr_destroy F
+GLIBC_2.25 pthread_condattr_init F
+GLIBC_2.25 pthread_equal F
+GLIBC_2.25 pthread_exit F
+GLIBC_2.25 pthread_getschedparam F
+GLIBC_2.25 pthread_mutex_destroy F
+GLIBC_2.25 pthread_mutex_init F
+GLIBC_2.25 pthread_mutex_lock F
+GLIBC_2.25 pthread_mutex_unlock F
+GLIBC_2.25 pthread_self F
+GLIBC_2.25 pthread_setcancelstate F
+GLIBC_2.25 pthread_setcanceltype F
+GLIBC_2.25 pthread_setschedparam F
+GLIBC_2.25 ptrace F
+GLIBC_2.25 ptsname F
+GLIBC_2.25 ptsname_r F
+GLIBC_2.25 putc F
+GLIBC_2.25 putc_unlocked F
+GLIBC_2.25 putchar F
+GLIBC_2.25 putchar_unlocked F
+GLIBC_2.25 putenv F
+GLIBC_2.25 putgrent F
+GLIBC_2.25 putmsg F
+GLIBC_2.25 putpmsg F
+GLIBC_2.25 putpwent F
+GLIBC_2.25 puts F
+GLIBC_2.25 putsgent F
+GLIBC_2.25 putspent F
+GLIBC_2.25 pututline F
+GLIBC_2.25 pututxline F
+GLIBC_2.25 putw F
+GLIBC_2.25 putwc F
+GLIBC_2.25 putwc_unlocked F
+GLIBC_2.25 putwchar F
+GLIBC_2.25 putwchar_unlocked F
+GLIBC_2.25 pvalloc F
+GLIBC_2.25 pwrite F
+GLIBC_2.25 pwrite64 F
+GLIBC_2.25 pwritev F
+GLIBC_2.25 pwritev64 F
+GLIBC_2.25 qecvt F
+GLIBC_2.25 qecvt_r F
+GLIBC_2.25 qfcvt F
+GLIBC_2.25 qfcvt_r F
+GLIBC_2.25 qgcvt F
+GLIBC_2.25 qsort F
+GLIBC_2.25 qsort_r F
+GLIBC_2.25 quick_exit F
+GLIBC_2.25 quotactl F
+GLIBC_2.25 raise F
+GLIBC_2.25 rand F
+GLIBC_2.25 rand_r F
+GLIBC_2.25 random F
+GLIBC_2.25 random_r F
+GLIBC_2.25 rawmemchr F
+GLIBC_2.25 rcmd F
+GLIBC_2.25 rcmd_af F
+GLIBC_2.25 re_comp F
+GLIBC_2.25 re_compile_fastmap F
+GLIBC_2.25 re_compile_pattern F
+GLIBC_2.25 re_exec F
+GLIBC_2.25 re_match F
+GLIBC_2.25 re_match_2 F
+GLIBC_2.25 re_search F
+GLIBC_2.25 re_search_2 F
+GLIBC_2.25 re_set_registers F
+GLIBC_2.25 re_set_syntax F
+GLIBC_2.25 re_syntax_options D 0x4
+GLIBC_2.25 read F
+GLIBC_2.25 readahead F
+GLIBC_2.25 readdir F
+GLIBC_2.25 readdir64 F
+GLIBC_2.25 readdir64_r F
+GLIBC_2.25 readdir_r F
+GLIBC_2.25 readlink F
+GLIBC_2.25 readlinkat F
+GLIBC_2.25 readv F
+GLIBC_2.25 realloc F
+GLIBC_2.25 realpath F
+GLIBC_2.25 reboot F
+GLIBC_2.25 recv F
+GLIBC_2.25 recvfrom F
+GLIBC_2.25 recvmmsg F
+GLIBC_2.25 recvmsg F
+GLIBC_2.25 regcomp F
+GLIBC_2.25 regerror F
+GLIBC_2.25 regexec F
+GLIBC_2.25 regfree F
+GLIBC_2.25 register_printf_function F
+GLIBC_2.25 register_printf_modifier F
+GLIBC_2.25 register_printf_specifier F
+GLIBC_2.25 register_printf_type F
+GLIBC_2.25 registerrpc F
+GLIBC_2.25 remap_file_pages F
+GLIBC_2.25 remove F
+GLIBC_2.25 removexattr F
+GLIBC_2.25 remque F
+GLIBC_2.25 rename F
+GLIBC_2.25 renameat F
+GLIBC_2.25 revoke F
+GLIBC_2.25 rewind F
+GLIBC_2.25 rewinddir F
+GLIBC_2.25 rexec F
+GLIBC_2.25 rexec_af F
+GLIBC_2.25 rexecoptions D 0x4
+GLIBC_2.25 rindex F
+GLIBC_2.25 rmdir F
+GLIBC_2.25 rpc_createerr D 0x10
+GLIBC_2.25 rpmatch F
+GLIBC_2.25 rresvport F
+GLIBC_2.25 rresvport_af F
+GLIBC_2.25 rtime F
+GLIBC_2.25 ruserok F
+GLIBC_2.25 ruserok_af F
+GLIBC_2.25 ruserpass F
+GLIBC_2.25 sbrk F
+GLIBC_2.25 scalbn F
+GLIBC_2.25 scalbnf F
+GLIBC_2.25 scalbnl F
+GLIBC_2.25 scandir F
+GLIBC_2.25 scandir64 F
+GLIBC_2.25 scandirat F
+GLIBC_2.25 scandirat64 F
+GLIBC_2.25 scanf F
+GLIBC_2.25 sched_get_priority_max F
+GLIBC_2.25 sched_get_priority_min F
+GLIBC_2.25 sched_getaffinity F
+GLIBC_2.25 sched_getcpu F
+GLIBC_2.25 sched_getparam F
+GLIBC_2.25 sched_getscheduler F
+GLIBC_2.25 sched_rr_get_interval F
+GLIBC_2.25 sched_setaffinity F
+GLIBC_2.25 sched_setparam F
+GLIBC_2.25 sched_setscheduler F
+GLIBC_2.25 sched_yield F
+GLIBC_2.25 secure_getenv F
+GLIBC_2.25 seed48 F
+GLIBC_2.25 seed48_r F
+GLIBC_2.25 seekdir F
+GLIBC_2.25 select F
+GLIBC_2.25 semctl F
+GLIBC_2.25 semget F
+GLIBC_2.25 semop F
+GLIBC_2.25 semtimedop F
+GLIBC_2.25 send F
+GLIBC_2.25 sendfile F
+GLIBC_2.25 sendfile64 F
+GLIBC_2.25 sendmmsg F
+GLIBC_2.25 sendmsg F
+GLIBC_2.25 sendto F
+GLIBC_2.25 setaliasent F
+GLIBC_2.25 setbuf F
+GLIBC_2.25 setbuffer F
+GLIBC_2.25 setcontext F
+GLIBC_2.25 setdomainname F
+GLIBC_2.25 setegid F
+GLIBC_2.25 setenv F
+GLIBC_2.25 seteuid F
+GLIBC_2.25 setfsent F
+GLIBC_2.25 setfsgid F
+GLIBC_2.25 setfsuid F
+GLIBC_2.25 setgid F
+GLIBC_2.25 setgrent F
+GLIBC_2.25 setgroups F
+GLIBC_2.25 sethostent F
+GLIBC_2.25 sethostid F
+GLIBC_2.25 sethostname F
+GLIBC_2.25 setipv4sourcefilter F
+GLIBC_2.25 setitimer F
+GLIBC_2.25 setjmp F
+GLIBC_2.25 setlinebuf F
+GLIBC_2.25 setlocale F
+GLIBC_2.25 setlogin F
+GLIBC_2.25 setlogmask F
+GLIBC_2.25 setmntent F
+GLIBC_2.25 setnetent F
+GLIBC_2.25 setnetgrent F
+GLIBC_2.25 setns F
+GLIBC_2.25 setpgid F
+GLIBC_2.25 setpgrp F
+GLIBC_2.25 setpriority F
+GLIBC_2.25 setprotoent F
+GLIBC_2.25 setpwent F
+GLIBC_2.25 setregid F
+GLIBC_2.25 setresgid F
+GLIBC_2.25 setresuid F
+GLIBC_2.25 setreuid F
+GLIBC_2.25 setrlimit F
+GLIBC_2.25 setrlimit64 F
+GLIBC_2.25 setrpcent F
+GLIBC_2.25 setservent F
+GLIBC_2.25 setsgent F
+GLIBC_2.25 setsid F
+GLIBC_2.25 setsockopt F
+GLIBC_2.25 setsourcefilter F
+GLIBC_2.25 setspent F
+GLIBC_2.25 setstate F
+GLIBC_2.25 setstate_r F
+GLIBC_2.25 settimeofday F
+GLIBC_2.25 setttyent F
+GLIBC_2.25 setuid F
+GLIBC_2.25 setusershell F
+GLIBC_2.25 setutent F
+GLIBC_2.25 setutxent F
+GLIBC_2.25 setvbuf F
+GLIBC_2.25 setxattr F
+GLIBC_2.25 sgetsgent F
+GLIBC_2.25 sgetsgent_r F
+GLIBC_2.25 sgetspent F
+GLIBC_2.25 sgetspent_r F
+GLIBC_2.25 shmat F
+GLIBC_2.25 shmctl F
+GLIBC_2.25 shmdt F
+GLIBC_2.25 shmget F
+GLIBC_2.25 shutdown F
+GLIBC_2.25 sigaction F
+GLIBC_2.25 sigaddset F
+GLIBC_2.25 sigaltstack F
+GLIBC_2.25 sigandset F
+GLIBC_2.25 sigblock F
+GLIBC_2.25 sigdelset F
+GLIBC_2.25 sigemptyset F
+GLIBC_2.25 sigfillset F
+GLIBC_2.25 siggetmask F
+GLIBC_2.25 sighold F
+GLIBC_2.25 sigignore F
+GLIBC_2.25 siginterrupt F
+GLIBC_2.25 sigisemptyset F
+GLIBC_2.25 sigismember F
+GLIBC_2.25 siglongjmp F
+GLIBC_2.25 signal F
+GLIBC_2.25 signalfd F
+GLIBC_2.25 sigorset F
+GLIBC_2.25 sigpause F
+GLIBC_2.25 sigpending F
+GLIBC_2.25 sigprocmask F
+GLIBC_2.25 sigqueue F
+GLIBC_2.25 sigrelse F
+GLIBC_2.25 sigreturn F
+GLIBC_2.25 sigset F
+GLIBC_2.25 sigsetmask F
+GLIBC_2.25 sigstack F
+GLIBC_2.25 sigsuspend F
+GLIBC_2.25 sigtimedwait F
+GLIBC_2.25 sigwait F
+GLIBC_2.25 sigwaitinfo F
+GLIBC_2.25 sleep F
+GLIBC_2.25 snprintf F
+GLIBC_2.25 sockatmark F
+GLIBC_2.25 socket F
+GLIBC_2.25 socketpair F
+GLIBC_2.25 splice F
+GLIBC_2.25 sprintf F
+GLIBC_2.25 sprofil F
+GLIBC_2.25 srand F
+GLIBC_2.25 srand48 F
+GLIBC_2.25 srand48_r F
+GLIBC_2.25 srandom F
+GLIBC_2.25 srandom_r F
+GLIBC_2.25 sscanf F
+GLIBC_2.25 ssignal F
+GLIBC_2.25 sstk F
+GLIBC_2.25 statfs F
+GLIBC_2.25 statfs64 F
+GLIBC_2.25 statvfs F
+GLIBC_2.25 statvfs64 F
+GLIBC_2.25 stderr D 0x4
+GLIBC_2.25 stdin D 0x4
+GLIBC_2.25 stdout D 0x4
+GLIBC_2.25 stime F
+GLIBC_2.25 stpcpy F
+GLIBC_2.25 stpncpy F
+GLIBC_2.25 strcasecmp F
+GLIBC_2.25 strcasecmp_l F
+GLIBC_2.25 strcasestr F
+GLIBC_2.25 strcat F
+GLIBC_2.25 strchr F
+GLIBC_2.25 strchrnul F
+GLIBC_2.25 strcmp F
+GLIBC_2.25 strcoll F
+GLIBC_2.25 strcoll_l F
+GLIBC_2.25 strcpy F
+GLIBC_2.25 strcspn F
+GLIBC_2.25 strdup F
+GLIBC_2.25 strerror F
+GLIBC_2.25 strerror_l F
+GLIBC_2.25 strerror_r F
+GLIBC_2.25 strfmon F
+GLIBC_2.25 strfmon_l F
+GLIBC_2.25 strfromd F
+GLIBC_2.25 strfromf F
+GLIBC_2.25 strfroml F
+GLIBC_2.25 strfry F
+GLIBC_2.25 strftime F
+GLIBC_2.25 strftime_l F
+GLIBC_2.25 strlen F
+GLIBC_2.25 strncasecmp F
+GLIBC_2.25 strncasecmp_l F
+GLIBC_2.25 strncat F
+GLIBC_2.25 strncmp F
+GLIBC_2.25 strncpy F
+GLIBC_2.25 strndup F
+GLIBC_2.25 strnlen F
+GLIBC_2.25 strpbrk F
+GLIBC_2.25 strptime F
+GLIBC_2.25 strptime_l F
+GLIBC_2.25 strrchr F
+GLIBC_2.25 strsep F
+GLIBC_2.25 strsignal F
+GLIBC_2.25 strspn F
+GLIBC_2.25 strstr F
+GLIBC_2.25 strtod F
+GLIBC_2.25 strtod_l F
+GLIBC_2.25 strtof F
+GLIBC_2.25 strtof_l F
+GLIBC_2.25 strtoimax F
+GLIBC_2.25 strtok F
+GLIBC_2.25 strtok_r F
+GLIBC_2.25 strtol F
+GLIBC_2.25 strtol_l F
+GLIBC_2.25 strtold F
+GLIBC_2.25 strtold_l F
+GLIBC_2.25 strtoll F
+GLIBC_2.25 strtoll_l F
+GLIBC_2.25 strtoq F
+GLIBC_2.25 strtoul F
+GLIBC_2.25 strtoul_l F
+GLIBC_2.25 strtoull F
+GLIBC_2.25 strtoull_l F
+GLIBC_2.25 strtoumax F
+GLIBC_2.25 strtouq F
+GLIBC_2.25 strverscmp F
+GLIBC_2.25 strxfrm F
+GLIBC_2.25 strxfrm_l F
+GLIBC_2.25 stty F
+GLIBC_2.25 svc_exit F
+GLIBC_2.25 svc_fdset D 0x80
+GLIBC_2.25 svc_getreq F
+GLIBC_2.25 svc_getreq_common F
+GLIBC_2.25 svc_getreq_poll F
+GLIBC_2.25 svc_getreqset F
+GLIBC_2.25 svc_max_pollfd D 0x4
+GLIBC_2.25 svc_pollfd D 0x4
+GLIBC_2.25 svc_register F
+GLIBC_2.25 svc_run F
+GLIBC_2.25 svc_sendreply F
+GLIBC_2.25 svc_unregister F
+GLIBC_2.25 svcauthdes_stats D 0xc
+GLIBC_2.25 svcerr_auth F
+GLIBC_2.25 svcerr_decode F
+GLIBC_2.25 svcerr_noproc F
+GLIBC_2.25 svcerr_noprog F
+GLIBC_2.25 svcerr_progvers F
+GLIBC_2.25 svcerr_systemerr F
+GLIBC_2.25 svcerr_weakauth F
+GLIBC_2.25 svcfd_create F
+GLIBC_2.25 svcraw_create F
+GLIBC_2.25 svctcp_create F
+GLIBC_2.25 svcudp_bufcreate F
+GLIBC_2.25 svcudp_create F
+GLIBC_2.25 svcudp_enablecache F
+GLIBC_2.25 svcunix_create F
+GLIBC_2.25 svcunixfd_create F
+GLIBC_2.25 swab F
+GLIBC_2.25 swapcontext F
+GLIBC_2.25 swapoff F
+GLIBC_2.25 swapon F
+GLIBC_2.25 swprintf F
+GLIBC_2.25 swscanf F
+GLIBC_2.25 symlink F
+GLIBC_2.25 symlinkat F
+GLIBC_2.25 sync F
+GLIBC_2.25 sync_file_range F
+GLIBC_2.25 syncfs F
+GLIBC_2.25 sys_errlist D 0x21c
+GLIBC_2.25 sys_nerr D 0x4
+GLIBC_2.25 sys_sigabbrev D 0x104
+GLIBC_2.25 sys_siglist D 0x104
+GLIBC_2.25 syscall F
+GLIBC_2.25 sysconf F
+GLIBC_2.25 sysctl F
+GLIBC_2.25 sysinfo F
+GLIBC_2.25 syslog F
+GLIBC_2.25 system F
+GLIBC_2.25 sysv_signal F
+GLIBC_2.25 tcdrain F
+GLIBC_2.25 tcflow F
+GLIBC_2.25 tcflush F
+GLIBC_2.25 tcgetattr F
+GLIBC_2.25 tcgetpgrp F
+GLIBC_2.25 tcgetsid F
+GLIBC_2.25 tcsendbreak F
+GLIBC_2.25 tcsetattr F
+GLIBC_2.25 tcsetpgrp F
+GLIBC_2.25 tdelete F
+GLIBC_2.25 tdestroy F
+GLIBC_2.25 tee F
+GLIBC_2.25 telldir F
+GLIBC_2.25 tempnam F
+GLIBC_2.25 textdomain F
+GLIBC_2.25 tfind F
+GLIBC_2.25 time F
+GLIBC_2.25 timegm F
+GLIBC_2.25 timelocal F
+GLIBC_2.25 timerfd_create F
+GLIBC_2.25 timerfd_gettime F
+GLIBC_2.25 timerfd_settime F
+GLIBC_2.25 times F
+GLIBC_2.25 timespec_get F
+GLIBC_2.25 timezone D 0x4
+GLIBC_2.25 tmpfile F
+GLIBC_2.25 tmpfile64 F
+GLIBC_2.25 tmpnam F
+GLIBC_2.25 tmpnam_r F
+GLIBC_2.25 toascii F
+GLIBC_2.25 tolower F
+GLIBC_2.25 tolower_l F
+GLIBC_2.25 toupper F
+GLIBC_2.25 toupper_l F
+GLIBC_2.25 towctrans F
+GLIBC_2.25 towctrans_l F
+GLIBC_2.25 towlower F
+GLIBC_2.25 towlower_l F
+GLIBC_2.25 towupper F
+GLIBC_2.25 towupper_l F
+GLIBC_2.25 tr_break F
+GLIBC_2.25 truncate F
+GLIBC_2.25 truncate64 F
+GLIBC_2.25 tsearch F
+GLIBC_2.25 ttyname F
+GLIBC_2.25 ttyname_r F
+GLIBC_2.25 ttyslot F
+GLIBC_2.25 twalk F
+GLIBC_2.25 tzname D 0x8
+GLIBC_2.25 tzset F
+GLIBC_2.25 ualarm F
+GLIBC_2.25 ulckpwdf F
+GLIBC_2.25 ulimit F
+GLIBC_2.25 umask F
+GLIBC_2.25 umount F
+GLIBC_2.25 umount2 F
+GLIBC_2.25 uname F
+GLIBC_2.25 ungetc F
+GLIBC_2.25 ungetwc F
+GLIBC_2.25 unlink F
+GLIBC_2.25 unlinkat F
+GLIBC_2.25 unlockpt F
+GLIBC_2.25 unsetenv F
+GLIBC_2.25 unshare F
+GLIBC_2.25 updwtmp F
+GLIBC_2.25 updwtmpx F
+GLIBC_2.25 uselocale F
+GLIBC_2.25 user2netname F
+GLIBC_2.25 usleep F
+GLIBC_2.25 ustat F
+GLIBC_2.25 utime F
+GLIBC_2.25 utimensat F
+GLIBC_2.25 utimes F
+GLIBC_2.25 utmpname F
+GLIBC_2.25 utmpxname F
+GLIBC_2.25 valloc F
+GLIBC_2.25 vasprintf F
+GLIBC_2.25 vdprintf F
+GLIBC_2.25 verr F
+GLIBC_2.25 verrx F
+GLIBC_2.25 versionsort F
+GLIBC_2.25 versionsort64 F
+GLIBC_2.25 vfork F
+GLIBC_2.25 vfprintf F
+GLIBC_2.25 vfscanf F
+GLIBC_2.25 vfwprintf F
+GLIBC_2.25 vfwscanf F
+GLIBC_2.25 vhangup F
+GLIBC_2.25 vlimit F
+GLIBC_2.25 vmsplice F
+GLIBC_2.25 vprintf F
+GLIBC_2.25 vscanf F
+GLIBC_2.25 vsnprintf F
+GLIBC_2.25 vsprintf F
+GLIBC_2.25 vsscanf F
+GLIBC_2.25 vswprintf F
+GLIBC_2.25 vswscanf F
+GLIBC_2.25 vsyslog F
+GLIBC_2.25 vtimes F
+GLIBC_2.25 vwarn F
+GLIBC_2.25 vwarnx F
+GLIBC_2.25 vwprintf F
+GLIBC_2.25 vwscanf F
+GLIBC_2.25 wait F
+GLIBC_2.25 wait3 F
+GLIBC_2.25 wait4 F
+GLIBC_2.25 waitid F
+GLIBC_2.25 waitpid F
+GLIBC_2.25 warn F
+GLIBC_2.25 warnx F
+GLIBC_2.25 wcpcpy F
+GLIBC_2.25 wcpncpy F
+GLIBC_2.25 wcrtomb F
+GLIBC_2.25 wcscasecmp F
+GLIBC_2.25 wcscasecmp_l F
+GLIBC_2.25 wcscat F
+GLIBC_2.25 wcschr F
+GLIBC_2.25 wcschrnul F
+GLIBC_2.25 wcscmp F
+GLIBC_2.25 wcscoll F
+GLIBC_2.25 wcscoll_l F
+GLIBC_2.25 wcscpy F
+GLIBC_2.25 wcscspn F
+GLIBC_2.25 wcsdup F
+GLIBC_2.25 wcsftime F
+GLIBC_2.25 wcsftime_l F
+GLIBC_2.25 wcslen F
+GLIBC_2.25 wcsncasecmp F
+GLIBC_2.25 wcsncasecmp_l F
+GLIBC_2.25 wcsncat F
+GLIBC_2.25 wcsncmp F
+GLIBC_2.25 wcsncpy F
+GLIBC_2.25 wcsnlen F
+GLIBC_2.25 wcsnrtombs F
+GLIBC_2.25 wcspbrk F
+GLIBC_2.25 wcsrchr F
+GLIBC_2.25 wcsrtombs F
+GLIBC_2.25 wcsspn F
+GLIBC_2.25 wcsstr F
+GLIBC_2.25 wcstod F
+GLIBC_2.25 wcstod_l F
+GLIBC_2.25 wcstof F
+GLIBC_2.25 wcstof_l F
+GLIBC_2.25 wcstoimax F
+GLIBC_2.25 wcstok F
+GLIBC_2.25 wcstol F
+GLIBC_2.25 wcstol_l F
+GLIBC_2.25 wcstold F
+GLIBC_2.25 wcstold_l F
+GLIBC_2.25 wcstoll F
+GLIBC_2.25 wcstoll_l F
+GLIBC_2.25 wcstombs F
+GLIBC_2.25 wcstoq F
+GLIBC_2.25 wcstoul F
+GLIBC_2.25 wcstoul_l F
+GLIBC_2.25 wcstoull F
+GLIBC_2.25 wcstoull_l F
+GLIBC_2.25 wcstoumax F
+GLIBC_2.25 wcstouq F
+GLIBC_2.25 wcswcs F
+GLIBC_2.25 wcswidth F
+GLIBC_2.25 wcsxfrm F
+GLIBC_2.25 wcsxfrm_l F
+GLIBC_2.25 wctob F
+GLIBC_2.25 wctomb F
+GLIBC_2.25 wctrans F
+GLIBC_2.25 wctrans_l F
+GLIBC_2.25 wctype F
+GLIBC_2.25 wctype_l F
+GLIBC_2.25 wcwidth F
+GLIBC_2.25 wmemchr F
+GLIBC_2.25 wmemcmp F
+GLIBC_2.25 wmemcpy F
+GLIBC_2.25 wmemmove F
+GLIBC_2.25 wmempcpy F
+GLIBC_2.25 wmemset F
+GLIBC_2.25 wordexp F
+GLIBC_2.25 wordfree F
+GLIBC_2.25 wprintf F
+GLIBC_2.25 write F
+GLIBC_2.25 writev F
+GLIBC_2.25 wscanf F
+GLIBC_2.25 xdecrypt F
+GLIBC_2.25 xdr_accepted_reply F
+GLIBC_2.25 xdr_array F
+GLIBC_2.25 xdr_authdes_cred F
+GLIBC_2.25 xdr_authdes_verf F
+GLIBC_2.25 xdr_authunix_parms F
+GLIBC_2.25 xdr_bool F
+GLIBC_2.25 xdr_bytes F
+GLIBC_2.25 xdr_callhdr F
+GLIBC_2.25 xdr_callmsg F
+GLIBC_2.25 xdr_char F
+GLIBC_2.25 xdr_cryptkeyarg F
+GLIBC_2.25 xdr_cryptkeyarg2 F
+GLIBC_2.25 xdr_cryptkeyres F
+GLIBC_2.25 xdr_des_block F
+GLIBC_2.25 xdr_double F
+GLIBC_2.25 xdr_enum F
+GLIBC_2.25 xdr_float F
+GLIBC_2.25 xdr_free F
+GLIBC_2.25 xdr_getcredres F
+GLIBC_2.25 xdr_hyper F
+GLIBC_2.25 xdr_int F
+GLIBC_2.25 xdr_int16_t F
+GLIBC_2.25 xdr_int32_t F
+GLIBC_2.25 xdr_int64_t F
+GLIBC_2.25 xdr_int8_t F
+GLIBC_2.25 xdr_key_netstarg F
+GLIBC_2.25 xdr_key_netstres F
+GLIBC_2.25 xdr_keybuf F
+GLIBC_2.25 xdr_keystatus F
+GLIBC_2.25 xdr_long F
+GLIBC_2.25 xdr_longlong_t F
+GLIBC_2.25 xdr_netnamestr F
+GLIBC_2.25 xdr_netobj F
+GLIBC_2.25 xdr_opaque F
+GLIBC_2.25 xdr_opaque_auth F
+GLIBC_2.25 xdr_pmap F
+GLIBC_2.25 xdr_pmaplist F
+GLIBC_2.25 xdr_pointer F
+GLIBC_2.25 xdr_quad_t F
+GLIBC_2.25 xdr_reference F
+GLIBC_2.25 xdr_rejected_reply F
+GLIBC_2.25 xdr_replymsg F
+GLIBC_2.25 xdr_rmtcall_args F
+GLIBC_2.25 xdr_rmtcallres F
+GLIBC_2.25 xdr_short F
+GLIBC_2.25 xdr_sizeof F
+GLIBC_2.25 xdr_string F
+GLIBC_2.25 xdr_u_char F
+GLIBC_2.25 xdr_u_hyper F
+GLIBC_2.25 xdr_u_int F
+GLIBC_2.25 xdr_u_long F
+GLIBC_2.25 xdr_u_longlong_t F
+GLIBC_2.25 xdr_u_quad_t F
+GLIBC_2.25 xdr_u_short F
+GLIBC_2.25 xdr_uint16_t F
+GLIBC_2.25 xdr_uint32_t F
+GLIBC_2.25 xdr_uint64_t F
+GLIBC_2.25 xdr_uint8_t F
+GLIBC_2.25 xdr_union F
+GLIBC_2.25 xdr_unixcred F
+GLIBC_2.25 xdr_vector F
+GLIBC_2.25 xdr_void F
+GLIBC_2.25 xdr_wrapstring F
+GLIBC_2.25 xdrmem_create F
+GLIBC_2.25 xdrrec_create F
+GLIBC_2.25 xdrrec_endofrecord F
+GLIBC_2.25 xdrrec_eof F
+GLIBC_2.25 xdrrec_skiprecord F
+GLIBC_2.25 xdrstdio_create F
+GLIBC_2.25 xencrypt F
+GLIBC_2.25 xprt_register F
+GLIBC_2.25 xprt_unregister F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libcrypt.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libcrypt.abilist
new file mode 100644
index 0000000000..309bc2c1f3
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libcrypt.abilist
@@ -0,0 +1,8 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 crypt F
+GLIBC_2.25 crypt_r F
+GLIBC_2.25 encrypt F
+GLIBC_2.25 encrypt_r F
+GLIBC_2.25 fcrypt F
+GLIBC_2.25 setkey F
+GLIBC_2.25 setkey_r F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist
new file mode 100644
index 0000000000..f635af5161
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libdl.abilist
@@ -0,0 +1,10 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 dladdr F
+GLIBC_2.25 dladdr1 F
+GLIBC_2.25 dlclose F
+GLIBC_2.25 dlerror F
+GLIBC_2.25 dlinfo F
+GLIBC_2.25 dlmopen F
+GLIBC_2.25 dlopen F
+GLIBC_2.25 dlsym F
+GLIBC_2.25 dlvsym F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist
new file mode 100644
index 0000000000..253ff25eb2
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libm.abilist
@@ -0,0 +1,455 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 _LIB_VERSION D 0x4
+GLIBC_2.25 __acos_finite F
+GLIBC_2.25 __acosf_finite F
+GLIBC_2.25 __acosh_finite F
+GLIBC_2.25 __acoshf_finite F
+GLIBC_2.25 __acoshl_finite F
+GLIBC_2.25 __acosl_finite F
+GLIBC_2.25 __asin_finite F
+GLIBC_2.25 __asinf_finite F
+GLIBC_2.25 __asinl_finite F
+GLIBC_2.25 __atan2_finite F
+GLIBC_2.25 __atan2f_finite F
+GLIBC_2.25 __atan2l_finite F
+GLIBC_2.25 __atanh_finite F
+GLIBC_2.25 __atanhf_finite F
+GLIBC_2.25 __atanhl_finite F
+GLIBC_2.25 __clog10 F
+GLIBC_2.25 __clog10f F
+GLIBC_2.25 __clog10l F
+GLIBC_2.25 __cosh_finite F
+GLIBC_2.25 __coshf_finite F
+GLIBC_2.25 __coshl_finite F
+GLIBC_2.25 __exp10_finite F
+GLIBC_2.25 __exp10f_finite F
+GLIBC_2.25 __exp10l_finite F
+GLIBC_2.25 __exp2_finite F
+GLIBC_2.25 __exp2f_finite F
+GLIBC_2.25 __exp2l_finite F
+GLIBC_2.25 __exp_finite F
+GLIBC_2.25 __expf_finite F
+GLIBC_2.25 __expl_finite F
+GLIBC_2.25 __finite F
+GLIBC_2.25 __finitef F
+GLIBC_2.25 __finitel F
+GLIBC_2.25 __fmod_finite F
+GLIBC_2.25 __fmodf_finite F
+GLIBC_2.25 __fmodl_finite F
+GLIBC_2.25 __fpclassify F
+GLIBC_2.25 __fpclassifyf F
+GLIBC_2.25 __fpclassifyl F
+GLIBC_2.25 __gamma_r_finite F
+GLIBC_2.25 __gammaf_r_finite F
+GLIBC_2.25 __gammal_r_finite F
+GLIBC_2.25 __hypot_finite F
+GLIBC_2.25 __hypotf_finite F
+GLIBC_2.25 __hypotl_finite F
+GLIBC_2.25 __iseqsig F
+GLIBC_2.25 __iseqsigf F
+GLIBC_2.25 __iseqsigl F
+GLIBC_2.25 __issignaling F
+GLIBC_2.25 __issignalingf F
+GLIBC_2.25 __issignalingl F
+GLIBC_2.25 __j0_finite F
+GLIBC_2.25 __j0f_finite F
+GLIBC_2.25 __j0l_finite F
+GLIBC_2.25 __j1_finite F
+GLIBC_2.25 __j1f_finite F
+GLIBC_2.25 __j1l_finite F
+GLIBC_2.25 __jn_finite F
+GLIBC_2.25 __jnf_finite F
+GLIBC_2.25 __jnl_finite F
+GLIBC_2.25 __lgamma_r_finite F
+GLIBC_2.25 __lgammaf_r_finite F
+GLIBC_2.25 __lgammal_r_finite F
+GLIBC_2.25 __log10_finite F
+GLIBC_2.25 __log10f_finite F
+GLIBC_2.25 __log10l_finite F
+GLIBC_2.25 __log2_finite F
+GLIBC_2.25 __log2f_finite F
+GLIBC_2.25 __log2l_finite F
+GLIBC_2.25 __log_finite F
+GLIBC_2.25 __logf_finite F
+GLIBC_2.25 __logl_finite F
+GLIBC_2.25 __pow_finite F
+GLIBC_2.25 __powf_finite F
+GLIBC_2.25 __powl_finite F
+GLIBC_2.25 __remainder_finite F
+GLIBC_2.25 __remainderf_finite F
+GLIBC_2.25 __remainderl_finite F
+GLIBC_2.25 __scalb_finite F
+GLIBC_2.25 __scalbf_finite F
+GLIBC_2.25 __scalbl_finite F
+GLIBC_2.25 __signbit F
+GLIBC_2.25 __signbitf F
+GLIBC_2.25 __signbitl F
+GLIBC_2.25 __signgam D 0x4
+GLIBC_2.25 __sinh_finite F
+GLIBC_2.25 __sinhf_finite F
+GLIBC_2.25 __sinhl_finite F
+GLIBC_2.25 __sqrt_finite F
+GLIBC_2.25 __sqrtf_finite F
+GLIBC_2.25 __sqrtl_finite F
+GLIBC_2.25 __y0_finite F
+GLIBC_2.25 __y0f_finite F
+GLIBC_2.25 __y0l_finite F
+GLIBC_2.25 __y1_finite F
+GLIBC_2.25 __y1f_finite F
+GLIBC_2.25 __y1l_finite F
+GLIBC_2.25 __yn_finite F
+GLIBC_2.25 __ynf_finite F
+GLIBC_2.25 __ynl_finite F
+GLIBC_2.25 acos F
+GLIBC_2.25 acosf F
+GLIBC_2.25 acosh F
+GLIBC_2.25 acoshf F
+GLIBC_2.25 acoshl F
+GLIBC_2.25 acosl F
+GLIBC_2.25 asin F
+GLIBC_2.25 asinf F
+GLIBC_2.25 asinh F
+GLIBC_2.25 asinhf F
+GLIBC_2.25 asinhl F
+GLIBC_2.25 asinl F
+GLIBC_2.25 atan F
+GLIBC_2.25 atan2 F
+GLIBC_2.25 atan2f F
+GLIBC_2.25 atan2l F
+GLIBC_2.25 atanf F
+GLIBC_2.25 atanh F
+GLIBC_2.25 atanhf F
+GLIBC_2.25 atanhl F
+GLIBC_2.25 atanl F
+GLIBC_2.25 cabs F
+GLIBC_2.25 cabsf F
+GLIBC_2.25 cabsl F
+GLIBC_2.25 cacos F
+GLIBC_2.25 cacosf F
+GLIBC_2.25 cacosh F
+GLIBC_2.25 cacoshf F
+GLIBC_2.25 cacoshl F
+GLIBC_2.25 cacosl F
+GLIBC_2.25 canonicalize F
+GLIBC_2.25 canonicalizef F
+GLIBC_2.25 canonicalizel F
+GLIBC_2.25 carg F
+GLIBC_2.25 cargf F
+GLIBC_2.25 cargl F
+GLIBC_2.25 casin F
+GLIBC_2.25 casinf F
+GLIBC_2.25 casinh F
+GLIBC_2.25 casinhf F
+GLIBC_2.25 casinhl F
+GLIBC_2.25 casinl F
+GLIBC_2.25 catan F
+GLIBC_2.25 catanf F
+GLIBC_2.25 catanh F
+GLIBC_2.25 catanhf F
+GLIBC_2.25 catanhl F
+GLIBC_2.25 catanl F
+GLIBC_2.25 cbrt F
+GLIBC_2.25 cbrtf F
+GLIBC_2.25 cbrtl F
+GLIBC_2.25 ccos F
+GLIBC_2.25 ccosf F
+GLIBC_2.25 ccosh F
+GLIBC_2.25 ccoshf F
+GLIBC_2.25 ccoshl F
+GLIBC_2.25 ccosl F
+GLIBC_2.25 ceil F
+GLIBC_2.25 ceilf F
+GLIBC_2.25 ceill F
+GLIBC_2.25 cexp F
+GLIBC_2.25 cexpf F
+GLIBC_2.25 cexpl F
+GLIBC_2.25 cimag F
+GLIBC_2.25 cimagf F
+GLIBC_2.25 cimagl F
+GLIBC_2.25 clog F
+GLIBC_2.25 clog10 F
+GLIBC_2.25 clog10f F
+GLIBC_2.25 clog10l F
+GLIBC_2.25 clogf F
+GLIBC_2.25 clogl F
+GLIBC_2.25 conj F
+GLIBC_2.25 conjf F
+GLIBC_2.25 conjl F
+GLIBC_2.25 copysign F
+GLIBC_2.25 copysignf F
+GLIBC_2.25 copysignl F
+GLIBC_2.25 cos F
+GLIBC_2.25 cosf F
+GLIBC_2.25 cosh F
+GLIBC_2.25 coshf F
+GLIBC_2.25 coshl F
+GLIBC_2.25 cosl F
+GLIBC_2.25 cpow F
+GLIBC_2.25 cpowf F
+GLIBC_2.25 cpowl F
+GLIBC_2.25 cproj F
+GLIBC_2.25 cprojf F
+GLIBC_2.25 cprojl F
+GLIBC_2.25 creal F
+GLIBC_2.25 crealf F
+GLIBC_2.25 creall F
+GLIBC_2.25 csin F
+GLIBC_2.25 csinf F
+GLIBC_2.25 csinh F
+GLIBC_2.25 csinhf F
+GLIBC_2.25 csinhl F
+GLIBC_2.25 csinl F
+GLIBC_2.25 csqrt F
+GLIBC_2.25 csqrtf F
+GLIBC_2.25 csqrtl F
+GLIBC_2.25 ctan F
+GLIBC_2.25 ctanf F
+GLIBC_2.25 ctanh F
+GLIBC_2.25 ctanhf F
+GLIBC_2.25 ctanhl F
+GLIBC_2.25 ctanl F
+GLIBC_2.25 drem F
+GLIBC_2.25 dremf F
+GLIBC_2.25 dreml F
+GLIBC_2.25 erf F
+GLIBC_2.25 erfc F
+GLIBC_2.25 erfcf F
+GLIBC_2.25 erfcl F
+GLIBC_2.25 erff F
+GLIBC_2.25 erfl F
+GLIBC_2.25 exp F
+GLIBC_2.25 exp10 F
+GLIBC_2.25 exp10f F
+GLIBC_2.25 exp10l F
+GLIBC_2.25 exp2 F
+GLIBC_2.25 exp2f F
+GLIBC_2.25 exp2l F
+GLIBC_2.25 expf F
+GLIBC_2.25 expl F
+GLIBC_2.25 expm1 F
+GLIBC_2.25 expm1f F
+GLIBC_2.25 expm1l F
+GLIBC_2.25 fabs F
+GLIBC_2.25 fabsf F
+GLIBC_2.25 fabsl F
+GLIBC_2.25 fdim F
+GLIBC_2.25 fdimf F
+GLIBC_2.25 fdiml F
+GLIBC_2.25 feclearexcept F
+GLIBC_2.25 fedisableexcept F
+GLIBC_2.25 feenableexcept F
+GLIBC_2.25 fegetenv F
+GLIBC_2.25 fegetexcept F
+GLIBC_2.25 fegetexceptflag F
+GLIBC_2.25 fegetmode F
+GLIBC_2.25 fegetround F
+GLIBC_2.25 feholdexcept F
+GLIBC_2.25 feraiseexcept F
+GLIBC_2.25 fesetenv F
+GLIBC_2.25 fesetexcept F
+GLIBC_2.25 fesetexceptflag F
+GLIBC_2.25 fesetmode F
+GLIBC_2.25 fesetround F
+GLIBC_2.25 fetestexcept F
+GLIBC_2.25 fetestexceptflag F
+GLIBC_2.25 feupdateenv F
+GLIBC_2.25 finite F
+GLIBC_2.25 finitef F
+GLIBC_2.25 finitel F
+GLIBC_2.25 floor F
+GLIBC_2.25 floorf F
+GLIBC_2.25 floorl F
+GLIBC_2.25 fma F
+GLIBC_2.25 fmaf F
+GLIBC_2.25 fmal F
+GLIBC_2.25 fmax F
+GLIBC_2.25 fmaxf F
+GLIBC_2.25 fmaxl F
+GLIBC_2.25 fmaxmag F
+GLIBC_2.25 fmaxmagf F
+GLIBC_2.25 fmaxmagl F
+GLIBC_2.25 fmin F
+GLIBC_2.25 fminf F
+GLIBC_2.25 fminl F
+GLIBC_2.25 fminmag F
+GLIBC_2.25 fminmagf F
+GLIBC_2.25 fminmagl F
+GLIBC_2.25 fmod F
+GLIBC_2.25 fmodf F
+GLIBC_2.25 fmodl F
+GLIBC_2.25 frexp F
+GLIBC_2.25 frexpf F
+GLIBC_2.25 frexpl F
+GLIBC_2.25 fromfp F
+GLIBC_2.25 fromfpf F
+GLIBC_2.25 fromfpl F
+GLIBC_2.25 fromfpx F
+GLIBC_2.25 fromfpxf F
+GLIBC_2.25 fromfpxl F
+GLIBC_2.25 gamma F
+GLIBC_2.25 gammaf F
+GLIBC_2.25 gammal F
+GLIBC_2.25 getpayload F
+GLIBC_2.25 getpayloadf F
+GLIBC_2.25 getpayloadl F
+GLIBC_2.25 hypot F
+GLIBC_2.25 hypotf F
+GLIBC_2.25 hypotl F
+GLIBC_2.25 ilogb F
+GLIBC_2.25 ilogbf F
+GLIBC_2.25 ilogbl F
+GLIBC_2.25 j0 F
+GLIBC_2.25 j0f F
+GLIBC_2.25 j0l F
+GLIBC_2.25 j1 F
+GLIBC_2.25 j1f F
+GLIBC_2.25 j1l F
+GLIBC_2.25 jn F
+GLIBC_2.25 jnf F
+GLIBC_2.25 jnl F
+GLIBC_2.25 ldexp F
+GLIBC_2.25 ldexpf F
+GLIBC_2.25 ldexpl F
+GLIBC_2.25 lgamma F
+GLIBC_2.25 lgamma_r F
+GLIBC_2.25 lgammaf F
+GLIBC_2.25 lgammaf_r F
+GLIBC_2.25 lgammal F
+GLIBC_2.25 lgammal_r F
+GLIBC_2.25 llogb F
+GLIBC_2.25 llogbf F
+GLIBC_2.25 llogbl F
+GLIBC_2.25 llrint F
+GLIBC_2.25 llrintf F
+GLIBC_2.25 llrintl F
+GLIBC_2.25 llround F
+GLIBC_2.25 llroundf F
+GLIBC_2.25 llroundl F
+GLIBC_2.25 log F
+GLIBC_2.25 log10 F
+GLIBC_2.25 log10f F
+GLIBC_2.25 log10l F
+GLIBC_2.25 log1p F
+GLIBC_2.25 log1pf F
+GLIBC_2.25 log1pl F
+GLIBC_2.25 log2 F
+GLIBC_2.25 log2f F
+GLIBC_2.25 log2l F
+GLIBC_2.25 logb F
+GLIBC_2.25 logbf F
+GLIBC_2.25 logbl F
+GLIBC_2.25 logf F
+GLIBC_2.25 logl F
+GLIBC_2.25 lrint F
+GLIBC_2.25 lrintf F
+GLIBC_2.25 lrintl F
+GLIBC_2.25 lround F
+GLIBC_2.25 lroundf F
+GLIBC_2.25 lroundl F
+GLIBC_2.25 matherr F
+GLIBC_2.25 modf F
+GLIBC_2.25 modff F
+GLIBC_2.25 modfl F
+GLIBC_2.25 nan F
+GLIBC_2.25 nanf F
+GLIBC_2.25 nanl F
+GLIBC_2.25 nearbyint F
+GLIBC_2.25 nearbyintf F
+GLIBC_2.25 nearbyintl F
+GLIBC_2.25 nextafter F
+GLIBC_2.25 nextafterf F
+GLIBC_2.25 nextafterl F
+GLIBC_2.25 nextdown F
+GLIBC_2.25 nextdownf F
+GLIBC_2.25 nextdownl F
+GLIBC_2.25 nexttoward F
+GLIBC_2.25 nexttowardf F
+GLIBC_2.25 nexttowardl F
+GLIBC_2.25 nextup F
+GLIBC_2.25 nextupf F
+GLIBC_2.25 nextupl F
+GLIBC_2.25 pow F
+GLIBC_2.25 pow10 F
+GLIBC_2.25 pow10f F
+GLIBC_2.25 pow10l F
+GLIBC_2.25 powf F
+GLIBC_2.25 powl F
+GLIBC_2.25 remainder F
+GLIBC_2.25 remainderf F
+GLIBC_2.25 remainderl F
+GLIBC_2.25 remquo F
+GLIBC_2.25 remquof F
+GLIBC_2.25 remquol F
+GLIBC_2.25 rint F
+GLIBC_2.25 rintf F
+GLIBC_2.25 rintl F
+GLIBC_2.25 round F
+GLIBC_2.25 roundeven F
+GLIBC_2.25 roundevenf F
+GLIBC_2.25 roundevenl F
+GLIBC_2.25 roundf F
+GLIBC_2.25 roundl F
+GLIBC_2.25 scalb F
+GLIBC_2.25 scalbf F
+GLIBC_2.25 scalbl F
+GLIBC_2.25 scalbln F
+GLIBC_2.25 scalblnf F
+GLIBC_2.25 scalblnl F
+GLIBC_2.25 scalbn F
+GLIBC_2.25 scalbnf F
+GLIBC_2.25 scalbnl F
+GLIBC_2.25 setpayload F
+GLIBC_2.25 setpayloadf F
+GLIBC_2.25 setpayloadl F
+GLIBC_2.25 setpayloadsig F
+GLIBC_2.25 setpayloadsigf F
+GLIBC_2.25 setpayloadsigl F
+GLIBC_2.25 signgam D 0x4
+GLIBC_2.25 significand F
+GLIBC_2.25 significandf F
+GLIBC_2.25 significandl F
+GLIBC_2.25 sin F
+GLIBC_2.25 sincos F
+GLIBC_2.25 sincosf F
+GLIBC_2.25 sincosl F
+GLIBC_2.25 sinf F
+GLIBC_2.25 sinh F
+GLIBC_2.25 sinhf F
+GLIBC_2.25 sinhl F
+GLIBC_2.25 sinl F
+GLIBC_2.25 sqrt F
+GLIBC_2.25 sqrtf F
+GLIBC_2.25 sqrtl F
+GLIBC_2.25 tan F
+GLIBC_2.25 tanf F
+GLIBC_2.25 tanh F
+GLIBC_2.25 tanhf F
+GLIBC_2.25 tanhl F
+GLIBC_2.25 tanl F
+GLIBC_2.25 tgamma F
+GLIBC_2.25 tgammaf F
+GLIBC_2.25 tgammal F
+GLIBC_2.25 totalorder F
+GLIBC_2.25 totalorderf F
+GLIBC_2.25 totalorderl F
+GLIBC_2.25 totalordermag F
+GLIBC_2.25 totalordermagf F
+GLIBC_2.25 totalordermagl F
+GLIBC_2.25 trunc F
+GLIBC_2.25 truncf F
+GLIBC_2.25 truncl F
+GLIBC_2.25 ufromfp F
+GLIBC_2.25 ufromfpf F
+GLIBC_2.25 ufromfpl F
+GLIBC_2.25 ufromfpx F
+GLIBC_2.25 ufromfpxf F
+GLIBC_2.25 ufromfpxl F
+GLIBC_2.25 y0 F
+GLIBC_2.25 y0f F
+GLIBC_2.25 y0l F
+GLIBC_2.25 y1 F
+GLIBC_2.25 y1f F
+GLIBC_2.25 y1l F
+GLIBC_2.25 yn F
+GLIBC_2.25 ynf F
+GLIBC_2.25 ynl F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist
new file mode 100644
index 0000000000..df234a66d4
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libnsl.abilist
@@ -0,0 +1,122 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 __free_fdresult F
+GLIBC_2.25 __nis_default_access F
+GLIBC_2.25 __nis_default_group F
+GLIBC_2.25 __nis_default_owner F
+GLIBC_2.25 __nis_default_ttl F
+GLIBC_2.25 __nis_finddirectory F
+GLIBC_2.25 __nis_hash F
+GLIBC_2.25 __nisbind_connect F
+GLIBC_2.25 __nisbind_create F
+GLIBC_2.25 __nisbind_destroy F
+GLIBC_2.25 __nisbind_next F
+GLIBC_2.25 __yp_check F
+GLIBC_2.25 nis_add F
+GLIBC_2.25 nis_add_entry F
+GLIBC_2.25 nis_addmember F
+GLIBC_2.25 nis_checkpoint F
+GLIBC_2.25 nis_clone_directory F
+GLIBC_2.25 nis_clone_object F
+GLIBC_2.25 nis_clone_result F
+GLIBC_2.25 nis_creategroup F
+GLIBC_2.25 nis_destroy_object F
+GLIBC_2.25 nis_destroygroup F
+GLIBC_2.25 nis_dir_cmp F
+GLIBC_2.25 nis_domain_of F
+GLIBC_2.25 nis_domain_of_r F
+GLIBC_2.25 nis_first_entry F
+GLIBC_2.25 nis_free_directory F
+GLIBC_2.25 nis_free_object F
+GLIBC_2.25 nis_free_request F
+GLIBC_2.25 nis_freenames F
+GLIBC_2.25 nis_freeresult F
+GLIBC_2.25 nis_freeservlist F
+GLIBC_2.25 nis_freetags F
+GLIBC_2.25 nis_getnames F
+GLIBC_2.25 nis_getservlist F
+GLIBC_2.25 nis_ismember F
+GLIBC_2.25 nis_leaf_of F
+GLIBC_2.25 nis_leaf_of_r F
+GLIBC_2.25 nis_lerror F
+GLIBC_2.25 nis_list F
+GLIBC_2.25 nis_local_directory F
+GLIBC_2.25 nis_local_group F
+GLIBC_2.25 nis_local_host F
+GLIBC_2.25 nis_local_principal F
+GLIBC_2.25 nis_lookup F
+GLIBC_2.25 nis_mkdir F
+GLIBC_2.25 nis_modify F
+GLIBC_2.25 nis_modify_entry F
+GLIBC_2.25 nis_name_of F
+GLIBC_2.25 nis_name_of_r F
+GLIBC_2.25 nis_next_entry F
+GLIBC_2.25 nis_perror F
+GLIBC_2.25 nis_ping F
+GLIBC_2.25 nis_print_directory F
+GLIBC_2.25 nis_print_entry F
+GLIBC_2.25 nis_print_group F
+GLIBC_2.25 nis_print_group_entry F
+GLIBC_2.25 nis_print_link F
+GLIBC_2.25 nis_print_object F
+GLIBC_2.25 nis_print_result F
+GLIBC_2.25 nis_print_rights F
+GLIBC_2.25 nis_print_table F
+GLIBC_2.25 nis_read_obj F
+GLIBC_2.25 nis_remove F
+GLIBC_2.25 nis_remove_entry F
+GLIBC_2.25 nis_removemember F
+GLIBC_2.25 nis_rmdir F
+GLIBC_2.25 nis_servstate F
+GLIBC_2.25 nis_sperrno F
+GLIBC_2.25 nis_sperror F
+GLIBC_2.25 nis_sperror_r F
+GLIBC_2.25 nis_stats F
+GLIBC_2.25 nis_verifygroup F
+GLIBC_2.25 nis_write_obj F
+GLIBC_2.25 readColdStartFile F
+GLIBC_2.25 writeColdStartFile F
+GLIBC_2.25 xdr_cback_data F
+GLIBC_2.25 xdr_domainname F
+GLIBC_2.25 xdr_keydat F
+GLIBC_2.25 xdr_mapname F
+GLIBC_2.25 xdr_obj_p F
+GLIBC_2.25 xdr_peername F
+GLIBC_2.25 xdr_valdat F
+GLIBC_2.25 xdr_yp_buf F
+GLIBC_2.25 xdr_ypall F
+GLIBC_2.25 xdr_ypbind_binding F
+GLIBC_2.25 xdr_ypbind_resp F
+GLIBC_2.25 xdr_ypbind_resptype F
+GLIBC_2.25 xdr_ypbind_setdom F
+GLIBC_2.25 xdr_ypdelete_args F
+GLIBC_2.25 xdr_ypmap_parms F
+GLIBC_2.25 xdr_ypmaplist F
+GLIBC_2.25 xdr_yppush_status F
+GLIBC_2.25 xdr_yppushresp_xfr F
+GLIBC_2.25 xdr_ypreq_key F
+GLIBC_2.25 xdr_ypreq_nokey F
+GLIBC_2.25 xdr_ypreq_xfr F
+GLIBC_2.25 xdr_ypresp_all F
+GLIBC_2.25 xdr_ypresp_key_val F
+GLIBC_2.25 xdr_ypresp_maplist F
+GLIBC_2.25 xdr_ypresp_master F
+GLIBC_2.25 xdr_ypresp_order F
+GLIBC_2.25 xdr_ypresp_val F
+GLIBC_2.25 xdr_ypresp_xfr F
+GLIBC_2.25 xdr_ypstat F
+GLIBC_2.25 xdr_ypupdate_args F
+GLIBC_2.25 xdr_ypxfrstat F
+GLIBC_2.25 yp_all F
+GLIBC_2.25 yp_bind F
+GLIBC_2.25 yp_first F
+GLIBC_2.25 yp_get_default_domain F
+GLIBC_2.25 yp_maplist F
+GLIBC_2.25 yp_master F
+GLIBC_2.25 yp_match F
+GLIBC_2.25 yp_next F
+GLIBC_2.25 yp_order F
+GLIBC_2.25 yp_unbind F
+GLIBC_2.25 yp_update F
+GLIBC_2.25 ypbinderr_string F
+GLIBC_2.25 yperr_string F
+GLIBC_2.25 ypprot_err F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist
new file mode 100644
index 0000000000..0808e139fa
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libpthread.abilist
@@ -0,0 +1,218 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 _IO_flockfile F
+GLIBC_2.25 _IO_ftrylockfile F
+GLIBC_2.25 _IO_funlockfile F
+GLIBC_2.25 __close F
+GLIBC_2.25 __connect F
+GLIBC_2.25 __errno_location F
+GLIBC_2.25 __fcntl F
+GLIBC_2.25 __h_errno_location F
+GLIBC_2.25 __libc_allocate_rtsig F
+GLIBC_2.25 __libc_current_sigrtmax F
+GLIBC_2.25 __libc_current_sigrtmin F
+GLIBC_2.25 __lseek F
+GLIBC_2.25 __nanosleep F
+GLIBC_2.25 __open F
+GLIBC_2.25 __open64 F
+GLIBC_2.25 __pread64 F
+GLIBC_2.25 __pthread_cleanup_routine F
+GLIBC_2.25 __pthread_getspecific F
+GLIBC_2.25 __pthread_key_create F
+GLIBC_2.25 __pthread_mutex_destroy F
+GLIBC_2.25 __pthread_mutex_init F
+GLIBC_2.25 __pthread_mutex_lock F
+GLIBC_2.25 __pthread_mutex_trylock F
+GLIBC_2.25 __pthread_mutex_unlock F
+GLIBC_2.25 __pthread_mutexattr_destroy F
+GLIBC_2.25 __pthread_mutexattr_init F
+GLIBC_2.25 __pthread_mutexattr_settype F
+GLIBC_2.25 __pthread_once F
+GLIBC_2.25 __pthread_register_cancel F
+GLIBC_2.25 __pthread_register_cancel_defer F
+GLIBC_2.25 __pthread_rwlock_destroy F
+GLIBC_2.25 __pthread_rwlock_init F
+GLIBC_2.25 __pthread_rwlock_rdlock F
+GLIBC_2.25 __pthread_rwlock_tryrdlock F
+GLIBC_2.25 __pthread_rwlock_trywrlock F
+GLIBC_2.25 __pthread_rwlock_unlock F
+GLIBC_2.25 __pthread_rwlock_wrlock F
+GLIBC_2.25 __pthread_setspecific F
+GLIBC_2.25 __pthread_unregister_cancel F
+GLIBC_2.25 __pthread_unregister_cancel_restore F
+GLIBC_2.25 __pthread_unwind_next F
+GLIBC_2.25 __pwrite64 F
+GLIBC_2.25 __read F
+GLIBC_2.25 __res_state F
+GLIBC_2.25 __send F
+GLIBC_2.25 __sigaction F
+GLIBC_2.25 __wait F
+GLIBC_2.25 __write F
+GLIBC_2.25 _pthread_cleanup_pop F
+GLIBC_2.25 _pthread_cleanup_pop_restore F
+GLIBC_2.25 _pthread_cleanup_push F
+GLIBC_2.25 _pthread_cleanup_push_defer F
+GLIBC_2.25 accept F
+GLIBC_2.25 close F
+GLIBC_2.25 connect F
+GLIBC_2.25 fcntl F
+GLIBC_2.25 flockfile F
+GLIBC_2.25 fsync F
+GLIBC_2.25 ftrylockfile F
+GLIBC_2.25 funlockfile F
+GLIBC_2.25 lseek F
+GLIBC_2.25 lseek64 F
+GLIBC_2.25 msync F
+GLIBC_2.25 nanosleep F
+GLIBC_2.25 open F
+GLIBC_2.25 open64 F
+GLIBC_2.25 pause F
+GLIBC_2.25 pread F
+GLIBC_2.25 pread64 F
+GLIBC_2.25 pthread_attr_destroy F
+GLIBC_2.25 pthread_attr_getaffinity_np F
+GLIBC_2.25 pthread_attr_getdetachstate F
+GLIBC_2.25 pthread_attr_getguardsize F
+GLIBC_2.25 pthread_attr_getinheritsched F
+GLIBC_2.25 pthread_attr_getschedparam F
+GLIBC_2.25 pthread_attr_getschedpolicy F
+GLIBC_2.25 pthread_attr_getscope F
+GLIBC_2.25 pthread_attr_getstack F
+GLIBC_2.25 pthread_attr_getstackaddr F
+GLIBC_2.25 pthread_attr_getstacksize F
+GLIBC_2.25 pthread_attr_init F
+GLIBC_2.25 pthread_attr_setaffinity_np F
+GLIBC_2.25 pthread_attr_setdetachstate F
+GLIBC_2.25 pthread_attr_setguardsize F
+GLIBC_2.25 pthread_attr_setinheritsched F
+GLIBC_2.25 pthread_attr_setschedparam F
+GLIBC_2.25 pthread_attr_setschedpolicy F
+GLIBC_2.25 pthread_attr_setscope F
+GLIBC_2.25 pthread_attr_setstack F
+GLIBC_2.25 pthread_attr_setstackaddr F
+GLIBC_2.25 pthread_attr_setstacksize F
+GLIBC_2.25 pthread_barrier_destroy F
+GLIBC_2.25 pthread_barrier_init F
+GLIBC_2.25 pthread_barrier_wait F
+GLIBC_2.25 pthread_barrierattr_destroy F
+GLIBC_2.25 pthread_barrierattr_getpshared F
+GLIBC_2.25 pthread_barrierattr_init F
+GLIBC_2.25 pthread_barrierattr_setpshared F
+GLIBC_2.25 pthread_cancel F
+GLIBC_2.25 pthread_cond_broadcast F
+GLIBC_2.25 pthread_cond_destroy F
+GLIBC_2.25 pthread_cond_init F
+GLIBC_2.25 pthread_cond_signal F
+GLIBC_2.25 pthread_cond_timedwait F
+GLIBC_2.25 pthread_cond_wait F
+GLIBC_2.25 pthread_condattr_destroy F
+GLIBC_2.25 pthread_condattr_getclock F
+GLIBC_2.25 pthread_condattr_getpshared F
+GLIBC_2.25 pthread_condattr_init F
+GLIBC_2.25 pthread_condattr_setclock F
+GLIBC_2.25 pthread_condattr_setpshared F
+GLIBC_2.25 pthread_create F
+GLIBC_2.25 pthread_detach F
+GLIBC_2.25 pthread_equal F
+GLIBC_2.25 pthread_exit F
+GLIBC_2.25 pthread_getaffinity_np F
+GLIBC_2.25 pthread_getattr_default_np F
+GLIBC_2.25 pthread_getattr_np F
+GLIBC_2.25 pthread_getconcurrency F
+GLIBC_2.25 pthread_getcpuclockid F
+GLIBC_2.25 pthread_getname_np F
+GLIBC_2.25 pthread_getschedparam F
+GLIBC_2.25 pthread_getspecific F
+GLIBC_2.25 pthread_join F
+GLIBC_2.25 pthread_key_create F
+GLIBC_2.25 pthread_key_delete F
+GLIBC_2.25 pthread_kill F
+GLIBC_2.25 pthread_kill_other_threads_np F
+GLIBC_2.25 pthread_mutex_consistent F
+GLIBC_2.25 pthread_mutex_consistent_np F
+GLIBC_2.25 pthread_mutex_destroy F
+GLIBC_2.25 pthread_mutex_getprioceiling F
+GLIBC_2.25 pthread_mutex_init F
+GLIBC_2.25 pthread_mutex_lock F
+GLIBC_2.25 pthread_mutex_setprioceiling F
+GLIBC_2.25 pthread_mutex_timedlock F
+GLIBC_2.25 pthread_mutex_trylock F
+GLIBC_2.25 pthread_mutex_unlock F
+GLIBC_2.25 pthread_mutexattr_destroy F
+GLIBC_2.25 pthread_mutexattr_getkind_np F
+GLIBC_2.25 pthread_mutexattr_getprioceiling F
+GLIBC_2.25 pthread_mutexattr_getprotocol F
+GLIBC_2.25 pthread_mutexattr_getpshared F
+GLIBC_2.25 pthread_mutexattr_getrobust F
+GLIBC_2.25 pthread_mutexattr_getrobust_np F
+GLIBC_2.25 pthread_mutexattr_gettype F
+GLIBC_2.25 pthread_mutexattr_init F
+GLIBC_2.25 pthread_mutexattr_setkind_np F
+GLIBC_2.25 pthread_mutexattr_setprioceiling F
+GLIBC_2.25 pthread_mutexattr_setprotocol F
+GLIBC_2.25 pthread_mutexattr_setpshared F
+GLIBC_2.25 pthread_mutexattr_setrobust F
+GLIBC_2.25 pthread_mutexattr_setrobust_np F
+GLIBC_2.25 pthread_mutexattr_settype F
+GLIBC_2.25 pthread_once F
+GLIBC_2.25 pthread_rwlock_destroy F
+GLIBC_2.25 pthread_rwlock_init F
+GLIBC_2.25 pthread_rwlock_rdlock F
+GLIBC_2.25 pthread_rwlock_timedrdlock F
+GLIBC_2.25 pthread_rwlock_timedwrlock F
+GLIBC_2.25 pthread_rwlock_tryrdlock F
+GLIBC_2.25 pthread_rwlock_trywrlock F
+GLIBC_2.25 pthread_rwlock_unlock F
+GLIBC_2.25 pthread_rwlock_wrlock F
+GLIBC_2.25 pthread_rwlockattr_destroy F
+GLIBC_2.25 pthread_rwlockattr_getkind_np F
+GLIBC_2.25 pthread_rwlockattr_getpshared F
+GLIBC_2.25 pthread_rwlockattr_init F
+GLIBC_2.25 pthread_rwlockattr_setkind_np F
+GLIBC_2.25 pthread_rwlockattr_setpshared F
+GLIBC_2.25 pthread_self F
+GLIBC_2.25 pthread_setaffinity_np F
+GLIBC_2.25 pthread_setattr_default_np F
+GLIBC_2.25 pthread_setcancelstate F
+GLIBC_2.25 pthread_setcanceltype F
+GLIBC_2.25 pthread_setconcurrency F
+GLIBC_2.25 pthread_setname_np F
+GLIBC_2.25 pthread_setschedparam F
+GLIBC_2.25 pthread_setschedprio F
+GLIBC_2.25 pthread_setspecific F
+GLIBC_2.25 pthread_sigmask F
+GLIBC_2.25 pthread_sigqueue F
+GLIBC_2.25 pthread_spin_destroy F
+GLIBC_2.25 pthread_spin_init F
+GLIBC_2.25 pthread_spin_lock F
+GLIBC_2.25 pthread_spin_trylock F
+GLIBC_2.25 pthread_spin_unlock F
+GLIBC_2.25 pthread_testcancel F
+GLIBC_2.25 pthread_timedjoin_np F
+GLIBC_2.25 pthread_tryjoin_np F
+GLIBC_2.25 pthread_yield F
+GLIBC_2.25 pwrite F
+GLIBC_2.25 pwrite64 F
+GLIBC_2.25 raise F
+GLIBC_2.25 read F
+GLIBC_2.25 recv F
+GLIBC_2.25 recvfrom F
+GLIBC_2.25 recvmsg F
+GLIBC_2.25 sem_close F
+GLIBC_2.25 sem_destroy F
+GLIBC_2.25 sem_getvalue F
+GLIBC_2.25 sem_init F
+GLIBC_2.25 sem_open F
+GLIBC_2.25 sem_post F
+GLIBC_2.25 sem_timedwait F
+GLIBC_2.25 sem_trywait F
+GLIBC_2.25 sem_unlink F
+GLIBC_2.25 sem_wait F
+GLIBC_2.25 send F
+GLIBC_2.25 sendmsg F
+GLIBC_2.25 sendto F
+GLIBC_2.25 sigaction F
+GLIBC_2.25 sigwait F
+GLIBC_2.25 tcdrain F
+GLIBC_2.25 wait F
+GLIBC_2.25 waitpid F
+GLIBC_2.25 write F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist
new file mode 100644
index 0000000000..27e62d1aeb
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libresolv.abilist
@@ -0,0 +1,82 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 __b64_ntop F
+GLIBC_2.25 __b64_pton F
+GLIBC_2.25 __dn_comp F
+GLIBC_2.25 __dn_count_labels F
+GLIBC_2.25 __dn_expand F
+GLIBC_2.25 __dn_skipname F
+GLIBC_2.25 __fp_nquery F
+GLIBC_2.25 __fp_query F
+GLIBC_2.25 __fp_resstat F
+GLIBC_2.25 __hostalias F
+GLIBC_2.25 __loc_aton F
+GLIBC_2.25 __loc_ntoa F
+GLIBC_2.25 __p_cdname F
+GLIBC_2.25 __p_cdnname F
+GLIBC_2.25 __p_class F
+GLIBC_2.25 __p_class_syms D 0x54
+GLIBC_2.25 __p_fqname F
+GLIBC_2.25 __p_fqnname F
+GLIBC_2.25 __p_option F
+GLIBC_2.25 __p_query F
+GLIBC_2.25 __p_rcode F
+GLIBC_2.25 __p_secstodate F
+GLIBC_2.25 __p_time F
+GLIBC_2.25 __p_type F
+GLIBC_2.25 __p_type_syms D 0x228
+GLIBC_2.25 __putlong F
+GLIBC_2.25 __putshort F
+GLIBC_2.25 __res_close F
+GLIBC_2.25 __res_dnok F
+GLIBC_2.25 __res_hnok F
+GLIBC_2.25 __res_hostalias F
+GLIBC_2.25 __res_isourserver F
+GLIBC_2.25 __res_mailok F
+GLIBC_2.25 __res_mkquery F
+GLIBC_2.25 __res_nameinquery F
+GLIBC_2.25 __res_nmkquery F
+GLIBC_2.25 __res_nquery F
+GLIBC_2.25 __res_nquerydomain F
+GLIBC_2.25 __res_nsearch F
+GLIBC_2.25 __res_nsend F
+GLIBC_2.25 __res_ownok F
+GLIBC_2.25 __res_queriesmatch F
+GLIBC_2.25 __res_query F
+GLIBC_2.25 __res_querydomain F
+GLIBC_2.25 __res_search F
+GLIBC_2.25 __res_send F
+GLIBC_2.25 __sym_ntop F
+GLIBC_2.25 __sym_ntos F
+GLIBC_2.25 __sym_ston F
+GLIBC_2.25 _getlong F
+GLIBC_2.25 _getshort F
+GLIBC_2.25 _res_opcodes D 0x40
+GLIBC_2.25 inet_net_ntop F
+GLIBC_2.25 inet_net_pton F
+GLIBC_2.25 inet_neta F
+GLIBC_2.25 ns_datetosecs F
+GLIBC_2.25 ns_format_ttl F
+GLIBC_2.25 ns_get16 F
+GLIBC_2.25 ns_get32 F
+GLIBC_2.25 ns_initparse F
+GLIBC_2.25 ns_makecanon F
+GLIBC_2.25 ns_msg_getflag F
+GLIBC_2.25 ns_name_compress F
+GLIBC_2.25 ns_name_ntol F
+GLIBC_2.25 ns_name_ntop F
+GLIBC_2.25 ns_name_pack F
+GLIBC_2.25 ns_name_pton F
+GLIBC_2.25 ns_name_rollback F
+GLIBC_2.25 ns_name_skip F
+GLIBC_2.25 ns_name_uncompress F
+GLIBC_2.25 ns_name_unpack F
+GLIBC_2.25 ns_parse_ttl F
+GLIBC_2.25 ns_parserr F
+GLIBC_2.25 ns_put16 F
+GLIBC_2.25 ns_put32 F
+GLIBC_2.25 ns_samedomain F
+GLIBC_2.25 ns_samename F
+GLIBC_2.25 ns_skiprr F
+GLIBC_2.25 ns_sprintrr F
+GLIBC_2.25 ns_sprintrrf F
+GLIBC_2.25 ns_subdomain F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist
new file mode 100644
index 0000000000..a94fd7f20e
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/librt.abilist
@@ -0,0 +1,36 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 __mq_open_2 F
+GLIBC_2.25 aio_cancel F
+GLIBC_2.25 aio_cancel64 F
+GLIBC_2.25 aio_error F
+GLIBC_2.25 aio_error64 F
+GLIBC_2.25 aio_fsync F
+GLIBC_2.25 aio_fsync64 F
+GLIBC_2.25 aio_init F
+GLIBC_2.25 aio_read F
+GLIBC_2.25 aio_read64 F
+GLIBC_2.25 aio_return F
+GLIBC_2.25 aio_return64 F
+GLIBC_2.25 aio_suspend F
+GLIBC_2.25 aio_suspend64 F
+GLIBC_2.25 aio_write F
+GLIBC_2.25 aio_write64 F
+GLIBC_2.25 lio_listio F
+GLIBC_2.25 lio_listio64 F
+GLIBC_2.25 mq_close F
+GLIBC_2.25 mq_getattr F
+GLIBC_2.25 mq_notify F
+GLIBC_2.25 mq_open F
+GLIBC_2.25 mq_receive F
+GLIBC_2.25 mq_send F
+GLIBC_2.25 mq_setattr F
+GLIBC_2.25 mq_timedreceive F
+GLIBC_2.25 mq_timedsend F
+GLIBC_2.25 mq_unlink F
+GLIBC_2.25 shm_open F
+GLIBC_2.25 shm_unlink F
+GLIBC_2.25 timer_create F
+GLIBC_2.25 timer_delete F
+GLIBC_2.25 timer_getoverrun F
+GLIBC_2.25 timer_gettime F
+GLIBC_2.25 timer_settime F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist
new file mode 100644
index 0000000000..9d6d31f3ef
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libthread_db.abilist
@@ -0,0 +1,41 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 td_init F
+GLIBC_2.25 td_log F
+GLIBC_2.25 td_symbol_list F
+GLIBC_2.25 td_ta_clear_event F
+GLIBC_2.25 td_ta_delete F
+GLIBC_2.25 td_ta_enable_stats F
+GLIBC_2.25 td_ta_event_addr F
+GLIBC_2.25 td_ta_event_getmsg F
+GLIBC_2.25 td_ta_get_nthreads F
+GLIBC_2.25 td_ta_get_ph F
+GLIBC_2.25 td_ta_get_stats F
+GLIBC_2.25 td_ta_map_id2thr F
+GLIBC_2.25 td_ta_map_lwp2thr F
+GLIBC_2.25 td_ta_new F
+GLIBC_2.25 td_ta_reset_stats F
+GLIBC_2.25 td_ta_set_event F
+GLIBC_2.25 td_ta_setconcurrency F
+GLIBC_2.25 td_ta_thr_iter F
+GLIBC_2.25 td_ta_tsd_iter F
+GLIBC_2.25 td_thr_clear_event F
+GLIBC_2.25 td_thr_dbresume F
+GLIBC_2.25 td_thr_dbsuspend F
+GLIBC_2.25 td_thr_event_enable F
+GLIBC_2.25 td_thr_event_getmsg F
+GLIBC_2.25 td_thr_get_info F
+GLIBC_2.25 td_thr_getfpregs F
+GLIBC_2.25 td_thr_getgregs F
+GLIBC_2.25 td_thr_getxregs F
+GLIBC_2.25 td_thr_getxregsize F
+GLIBC_2.25 td_thr_set_event F
+GLIBC_2.25 td_thr_setfpregs F
+GLIBC_2.25 td_thr_setgregs F
+GLIBC_2.25 td_thr_setprio F
+GLIBC_2.25 td_thr_setsigpending F
+GLIBC_2.25 td_thr_setxregs F
+GLIBC_2.25 td_thr_sigsetmask F
+GLIBC_2.25 td_thr_tls_get_addr F
+GLIBC_2.25 td_thr_tlsbase F
+GLIBC_2.25 td_thr_tsd F
+GLIBC_2.25 td_thr_validate F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist b/sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist
new file mode 100644
index 0000000000..140f5b8a10
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/aarch64/ilp32/libutil.abilist
@@ -0,0 +1,7 @@
+GLIBC_2.25 GLIBC_2.25 A
+GLIBC_2.25 forkpty F
+GLIBC_2.25 login F
+GLIBC_2.25 login_tty F
+GLIBC_2.25 logout F
+GLIBC_2.25 logwtmp F
+GLIBC_2.25 openpty F
diff --git a/sysdeps/unix/sysv/linux/aarch64/ld.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/ld.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/ld.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/ld.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libBrokenLocale.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libBrokenLocale.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libBrokenLocale.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libBrokenLocale.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libanl.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libanl.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libanl.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libanl.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libc.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libc.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libc.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libc.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libcrypt.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libcrypt.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libcrypt.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libcrypt.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libdl.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libdl.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libdl.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libdl.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libm.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libm.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libm.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libm.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libnsl.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libnsl.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libnsl.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libnsl.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libpthread.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libpthread.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libpthread.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libpthread.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libresolv.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libresolv.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libresolv.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libresolv.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/librt.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/librt.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/librt.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/librt.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libthread_db.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libthread_db.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libthread_db.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libthread_db.abilist
diff --git a/sysdeps/unix/sysv/linux/aarch64/libutil.abilist b/sysdeps/unix/sysv/linux/aarch64/lp64/libutil.abilist
similarity index 100%
rename from sysdeps/unix/sysv/linux/aarch64/libutil.abilist
rename to sysdeps/unix/sysv/linux/aarch64/lp64/libutil.abilist
-- 
2.13.0

